From 3c0389e4322799089ac4479b1f584a42aac03948 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 6 May 2019 21:22:15 -0600
Subject: [PATCH] [SQUASH][DNM] Merge tag 'android-9.0.0_r37' into
 staging/lineage-16.0_merge-android-9.0.0_r37

Android 9.0.0 Release 37 (PQ3A.190505.002)

* tag 'android-9.0.0_r37':
  Import translations. DO NOT MERGE
  Fixed that emergency alert not displayed when screen saver is on
  Import translations. DO NOT MERGE
  Fixed that extra vibration was played

https://github.com/haggertk/android_packages_apps_CellBroadcastReceiver/tree/lineage-16.0_merge-android-9.0.0_r37

Change-Id: Ic3eb4c69366c026b1017b6b687819d7e8252a556
---
 AndroidManifest.xml                           |   1 +
 res/values-ar/strings.xml                     |   2 +-
 res/values-as/strings.xml                     |  11 +-
 res/values-da/strings.xml                     |   8 +-
 res/values-en-rXC/strings.xml                 | 224 +++++++++---------
 res/values-gl/strings.xml                     |  36 +--
 res/values-ko/strings.xml                     |   2 +-
 res/values-mcc246-en-rXC/strings.xml          |   6 +-
 res/values-mcc450-en-rXC/strings.xml          |   2 +-
 res/values-mcc450-gl/strings.xml              |   2 +-
 res/values-mcc466-en-rXC/strings.xml          |   2 +-
 res/values-mcc716-en-rXC/strings.xml          |   4 +-
 res/values-nl/strings.xml                     |   2 +-
 .../CellBroadcastAlertAudio.java              |  10 +-
 .../CellBroadcastAlertService.java            |   6 +
 15 files changed, 164 insertions(+), 154 deletions(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index adbe01b5..8b98198f 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -30,6 +30,7 @@
     <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.INTERACT_ACROSS_USERS" />
     <uses-permission android:name="android.permission.MANAGE_USERS" />
+    <uses-permission android:name="android.permission.DEVICE_POWER" />
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
 
     <application android:name="CellBroadcastReceiverApp"
diff --git a/res/values-ar/strings.xml b/res/values-ar/strings.xml
index 88a9e79f..51008e32 100644
--- a/res/values-ar/strings.xml
+++ b/res/values-ar/strings.xml
@@ -127,7 +127,7 @@
     <item msgid="9097229303902157183">"كل دقيقتين"</item>
     <item msgid="5718214950343391480">"كل ٥ دقائق"</item>
     <item msgid="3863339891188103437">"كل ١٥ دقيقة"</item>
-    <item msgid="6868848414437854609">"إيقاف"</item>
+    <item msgid="6868848414437854609">"غير مفعّل"</item>
   </string-array>
     <string name="emergency_alert_settings_title_watches" msgid="4477073412799894883">"إنذارات الطوارئ اللاسلكية"</string>
 </resources>
diff --git a/res/values-as/strings.xml b/res/values-as/strings.xml
index ff8036ae..0690bcd3 100644
--- a/res/values-as/strings.xml
+++ b/res/values-as/strings.xml
@@ -46,8 +46,7 @@
     <string name="cmas_operator_defined_alert" msgid="8755372450810011476">"জৰুৰীকালীন সতৰ্কবাণী (অপাৰেটৰ)"</string>
     <string name="cb_other_message_identifiers" msgid="5790068194529377210">"সম্প্ৰচাৰ কৰা বাৰ্তাবোৰ"</string>
     <string name="public_safety_message" msgid="9119928798786998252">"ৰাজহুৱা নিৰাপত্তা বিষয়ক বাৰ্তা"</string>
-    <!-- no translation found for state_local_test_alert (8003145745857480200) -->
-    <skip />
+    <string name="state_local_test_alert" msgid="8003145745857480200">"ৰাজ্যিক/স্থানীয় পৰীক্ষামূলক সতৰ্কবাণী"</string>
     <string name="emergency_alert" msgid="624783871477634263">"জৰুৰীকালীন সতৰ্কবাণী"</string>
     <string name="emergency_alerts_title" msgid="6605036374197485429">"সতৰ্কবাণীসমূহ"</string>
     <string name="notification_channel_broadcast_messages" msgid="880704362482824524">"সম্প্ৰচাৰ কৰা বাৰ্তাবোৰ"</string>
@@ -71,10 +70,8 @@
     <string name="enable_alert_message_summary" msgid="6525664541696985610">"আসন্ন বিপদাশংকাৰ বিষয়ে সতৰ্ক কৰক"</string>
     <string name="enable_public_safety_messages_title" msgid="5576770949182656524">"ৰাজহুৱা নিৰাপত্তা বিষয়ক বাৰ্তাসমূহ"</string>
     <string name="enable_public_safety_messages_summary" msgid="7868069748857851521">"জীৱন বা সম্পত্তি ৰক্ষা কৰিব পৰা অনুমোদিত কাৰ্যসমূহ"</string>
-    <!-- no translation found for enable_state_local_test_alerts_title (1012930918171302720) -->
-    <skip />
-    <!-- no translation found for enable_state_local_test_alerts_summary (780298327377950187) -->
-    <skip />
+    <string name="enable_state_local_test_alerts_title" msgid="1012930918171302720">"ৰাজ্যিক আৰু স্থানীয় পৰীক্ষামূলক সতৰ্কবাণী"</string>
+    <string name="enable_state_local_test_alerts_summary" msgid="780298327377950187">"ৰাজ্য চৰকাৰ আৰু স্থানীয় কৰ্তৃপক্ষৰ পৰা পৰীক্ষামূলক বাৰ্তা লাভ কৰক"</string>
     <string name="enable_emergency_alerts_message_title" msgid="661894007489847468">"জৰুৰীকালীন সতৰ্কবাণীসমূহ"</string>
     <string name="enable_emergency_alerts_message_summary" msgid="7574617515441602546">"জীৱন-সংশয়ী ঘটনাসমূহৰ বিষয়ে সতৰ্ক কৰক"</string>
     <string name="enable_cmas_test_alerts_title" msgid="1319231576046146463">"দৰকাৰী মাহেকীয়া পৰীক্ষা"</string>
@@ -82,7 +79,7 @@
     <string name="enable_alert_vibrate_title" msgid="982817538757982090">"কম্পন"</string>
     <string name="use_full_volume_title" msgid="8581439612945182255">"সম্পূৰ্ণ ভলিউম ব্য়ৱহাৰ কৰক"</string>
     <string name="use_full_volume_summary" msgid="3560764277281704697">"অন্য়ান্য় ভলিউম আৰু \'অসুবিধা নিদিব\'ৰ অগ্ৰাধিকাৰবোৰ উপেক্ষা কৰক"</string>
-    <string name="enable_area_update_info_alerts_title" msgid="3442042268424617226">"এলেকা আপডেট সম্প্ৰচাৰসমূহ"</string>
+    <string name="enable_area_update_info_alerts_title" msgid="3442042268424617226">"এলেকা আপডে’ট সম্প্ৰচাৰসমূহ"</string>
     <string name="enable_area_update_info_alerts_summary" msgid="6437816607144264910">"ছিমৰ স্থিতিত আপডেটৰ তথ্য় দেখুৱাওক"</string>
     <string name="category_dev_settings_title" msgid="6194393458398329994">"বিকাশকৰ্তাৰ বিকল্পসমূহ"</string>
     <string name="cmas_category_heading" msgid="3923503130776640717">"সতৰ্কবাণীৰ প্ৰকাৰ:"</string>
diff --git a/res/values-da/strings.xml b/res/values-da/strings.xml
index 12cef8b1..12d2787b 100644
--- a/res/values-da/strings.xml
+++ b/res/values-da/strings.xml
@@ -23,9 +23,9 @@
     <string name="no_cell_broadcasts" msgid="5409324337492758562">"Der er ingen aktive advarsler i dit område. Du kan ændre indstillingerne for advarsler i menuen Indstillinger."</string>
     <string name="menu_delete_all" msgid="3940997343921149800">"Slet meddelelser"</string>
     <string name="message_options" msgid="3178489901903589574">"Valgmuligheder for beskeder"</string>
-    <string name="menu_view_details" msgid="1040989019045280975">"Se detaljer"</string>
+    <string name="menu_view_details" msgid="1040989019045280975">"Se info"</string>
     <string name="menu_delete" msgid="128380070910799366">"Slet meddelelse"</string>
-    <string name="view_details_title" msgid="1780427629491781473">"Detaljer om underretning"</string>
+    <string name="view_details_title" msgid="1780427629491781473">"Detaljer om advarsel"</string>
     <string name="confirm_delete_broadcast" msgid="2540199303730232322">"Vil du slette denne meddelelse?"</string>
     <string name="confirm_delete_all_broadcasts" msgid="2924444089047280871">"Vil du slette alle modtagne meddelelser?"</string>
     <string name="button_delete" msgid="4672451757925194350">"Slet"</string>
@@ -52,7 +52,7 @@
     <string name="notification_channel_broadcast_messages" msgid="880704362482824524">"Udsendte beskeder"</string>
     <string name="notification_channel_emergency_alerts" msgid="5008287980979183617">"Nødalarmer"</string>
     <string name="enable_alerts_master_toggle_title" msgid="1457904343636699446">"Tillad underretninger"</string>
-    <string name="enable_alerts_master_toggle_summary" msgid="699552922381916044">"Få underretninger i nødsituationer"</string>
+    <string name="enable_alerts_master_toggle_summary" msgid="699552922381916044">"Få notifikationer i nødsituationer"</string>
     <string name="alert_reminder_interval_title" msgid="7466642011937564868">"Lyd for alarmpåmindelse"</string>
     <string name="enable_alert_speech_title" msgid="8052104771053526941">"Læs advarselsmeddelelser op"</string>
     <string name="enable_alert_speech_summary" msgid="356086178962268831">"Brug Oplæsning for at indtale beskeder om nødalarmer"</string>
@@ -82,7 +82,7 @@
     <string name="enable_area_update_info_alerts_title" msgid="3442042268424617226">"Meddelelser om områdeopdatering"</string>
     <string name="enable_area_update_info_alerts_summary" msgid="6437816607144264910">"Vis opdateringsoplysninger i SIM-status"</string>
     <string name="category_dev_settings_title" msgid="6194393458398329994">"Indstillinger for udviklere"</string>
-    <string name="cmas_category_heading" msgid="3923503130776640717">"Underretningskategori:"</string>
+    <string name="cmas_category_heading" msgid="3923503130776640717">"Advarselskategori:"</string>
     <string name="cmas_category_geo" msgid="4979494217069688527">"Geofysisk"</string>
     <string name="cmas_category_met" msgid="7563732573851773537">"Meteorologisk"</string>
     <string name="cmas_category_safety" msgid="2986472639641883453">"Sikkerhed"</string>
diff --git a/res/values-en-rXC/strings.xml b/res/values-en-rXC/strings.xml
index 67952356..ddfdcbe5 100644
--- a/res/values-en-rXC/strings.xml
+++ b/res/values-en-rXC/strings.xml
@@ -16,118 +16,118 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="app_label" msgid="7559008249836700599">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‎‎‎‏‏‏‎‎‏‏‎‏‏‏‏‏‏‎‏‏‏‏‏‎‏‎‎‎‎‏‏‎‎‎‏‏‏‎‎‎‎‏‏‏‎‎‎‎‎‏‏‏‎‏‏‎‏‏‏‎Cell Broadcasts‎‏‎‎‏‎"</string>
-    <string name="sms_cb_settings" msgid="3795852619005211509">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‏‎‎‏‎‏‎‏‏‎‏‏‎‎‏‎‎‏‏‏‏‎‏‏‎‎‎‎‏‏‏‏‏‏‏‏‎‎‎‏‎‏‎‏‎‏‏‎‎‏‏‎‏‏‏‎‏‎‏‎Emergency alerts‎‏‎‎‏‎"</string>
-    <string name="cell_broadcast_settings_not_available" msgid="2612364350745967915">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‎‏‎‎‎‏‎‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‎‎‏‎‎‏‎‎‏‏‏‏‏‎‎‎‎‏‏‎‏‏‎‏‏‏‏‏‎‏‎‎‏‎‏‎‏‏‎Cell Broadcast settings are not available for this user‎‏‎‎‏‎"</string>
-    <string name="button_dismiss" msgid="1234221657930516287">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‎‎‏‎‎‏‎‎‎‎‎‏‏‎‏‎‏‏‎‎‎‎‎‏‏‏‏‎‎‎‎‎‏‏‏‎‎‎‎‏‎‏‎‏‏‎‎‎‏‏‏‎‎‏‏‏‏‏‏‎OK‎‏‎‎‏‎"</string>
-    <string name="no_cell_broadcasts" msgid="5409324337492758562">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‎‏‏‎‎‎‏‎‎‎‏‏‏‎‎‎‏‏‏‏‏‎‏‎‎‏‏‎‎‏‏‏‎‏‏‎‎‏‎‎‎‎‏‏‎‎‏‎‎‎‎‎‎‏‎‎‎‏‎‎There are no active alerts in your area. You can change the Alert settings using the Settings menu option.‎‏‎‎‏‎"</string>
-    <string name="menu_delete_all" msgid="3940997343921149800">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‏‏‎‏‎‏‏‎‎‎‏‎‎‏‏‏‏‎‎‎‎‏‏‎‏‎‎‎‎‏‏‏‎‎‏‎‎‏‎‏‏‎‏‏‎‎‏‎‏‏‏‎‏‏‎‏‎‎‎‎Delete broadcasts‎‏‎‎‏‎"</string>
-    <string name="message_options" msgid="3178489901903589574">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‏‎‎‎‎‎‏‏‏‎‎‎‏‎‎‎‎‏‏‏‎‏‏‏‎‎‏‏‎‏‏‎‏‎‏‎‎‏‎‎‎‎‏‎‎‏‏‎‎‎‎‏‏‎‎‎‏‏‎‎Message options‎‏‎‎‏‎"</string>
-    <string name="menu_view_details" msgid="1040989019045280975">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‏‎‎‏‏‏‎‎‏‎‎‏‎‏‎‏‎‏‏‏‏‏‏‏‏‏‎‏‎‏‎‏‏‏‎‏‎‎‎‏‏‎‏‎‏‏‏‏‎‎‏‏‎‎‏‏‏‏‎View details‎‏‎‎‏‎"</string>
-    <string name="menu_delete" msgid="128380070910799366">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‎‏‏‏‏‎‎‏‎‎‎‎‎‎‏‏‎‎‎‏‏‏‏‏‏‏‎‏‎‎‎‏‎‏‎‏‏‏‎‎‎‎‎‏‎‏‎‏‏‏‎‎‎‎‎‎‏‏‎‎Delete broadcast‎‏‎‎‏‎"</string>
-    <string name="view_details_title" msgid="1780427629491781473">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‏‎‎‎‏‎‏‏‎‏‎‏‎‏‎‏‏‎‎‏‎‏‏‏‏‏‏‎‏‏‏‏‎‏‎‎‎‎‏‎‎‏‎‏‏‏‏‏‏‏‏‏‎‏‏‎‎‎‎‏‎Alert details‎‏‎‎‏‎"</string>
-    <string name="confirm_delete_broadcast" msgid="2540199303730232322">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‎‎‏‏‎‏‎‎‎‎‎‎‏‎‎‏‏‎‎‏‏‏‎‎‏‏‏‎‏‏‏‎‎‏‎‏‏‎‎‎‏‎‎‎‏‏‏‏‏‏‎‎‎‎‎‎‎‎‏‎‎Delete this broadcast?‎‏‎‎‏‎"</string>
-    <string name="confirm_delete_all_broadcasts" msgid="2924444089047280871">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‎‎‏‎‎‏‎‏‎‏‏‎‏‏‎‏‏‎‎‏‏‎‎‏‏‏‏‎‎‏‎‏‎‏‎‎‏‎‎‏‏‏‎‏‏‎‏‏‎‎‏‏‏‎‎‏‏‏‎Delete all received broadcast messages?‎‏‎‎‏‎"</string>
-    <string name="button_delete" msgid="4672451757925194350">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‎‎‎‏‏‎‏‎‏‏‏‏‏‏‎‎‎‏‎‎‎‎‏‏‏‏‎‎‎‏‏‏‏‎‏‎‏‎‎‎‏‏‏‏‏‎‏‏‎‏‎‎‏‏‎‏‏‏‎‎Delete‎‏‎‎‏‎"</string>
-    <string name="button_cancel" msgid="7479958360523246140">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‏‏‏‏‎‎‏‏‏‎‎‎‏‎‎‏‏‎‎‏‏‏‏‏‏‏‏‎‎‏‎‏‎‏‏‎‏‏‎‎‏‏‏‎‎‎‏‎‏‎‎‎‏‏‏‏‎‎‎Cancel‎‏‎‎‏‎"</string>
-    <string name="etws_earthquake_warning" msgid="6428741104423152511">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‎‏‎‎‏‏‎‏‏‏‎‏‏‏‏‎‎‏‏‏‏‏‏‎‎‏‎‎‏‏‏‎‏‎‎‎‎‏‏‎‎‎‎‎‏‎‏‎‏‏‎‏‏‏‏‏‏‏‎Earthquake warning‎‏‎‎‏‎"</string>
-    <string name="etws_tsunami_warning" msgid="6173964105145900312">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‏‎‏‏‎‏‎‏‏‏‎‎‏‎‏‎‎‏‏‏‎‏‎‎‏‎‎‎‏‏‏‏‏‏‏‎‎‏‏‏‎‏‏‏‏‎‏‏‏‎‏‎‎‎‏‏‎‎‎‎Tsunami warning‎‏‎‎‏‎"</string>
-    <string name="etws_earthquake_and_tsunami_warning" msgid="662449983177407681">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‏‎‎‏‏‎‎‎‏‎‏‏‏‏‏‏‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‏‎‎‎‎‎‎‏‎‏‏‏‏‎‎‏‏‎‎‎‎‎‏‎Earthquake and tsunami warning‎‏‎‎‏‎"</string>
-    <string name="etws_test_message" msgid="8447820262584381894">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‎‏‎‎‏‏‏‏‎‎‏‎‏‎‏‏‏‏‏‏‎‎‎‏‎‏‎‎‏‏‎‏‎‏‏‎‏‏‎‎‎‎‎‏‏‏‎‎‎‏‏‏‎‎‎‏‏‎‎ETWS test message‎‏‎‎‏‎"</string>
-    <string name="etws_other_emergency_type" msgid="5233080551309721499">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‎‎‎‏‎‎‏‏‏‏‏‏‎‏‎‎‎‏‏‎‎‎‎‏‎‎‏‎‎‎‎‏‏‎‏‏‏‎‏‎‎‏‎‎‎‏‏‏‏‏‏‏‎‎‏‏‎‏‏‎Emergency warning‎‏‎‎‏‎"</string>
-    <string name="cmas_presidential_level_alert" msgid="1209234030582361001">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‎‎‎‏‏‎‎‏‎‎‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‎‏‏‎‏‎‏‎‏‏‎‎‏‎‏‏‎‎‏‏‏‎‏‎‏‏‏‎‏‎‏‎‎‏‎Presidential alert‎‏‎‎‏‎"</string>
-    <string name="cmas_extreme_alert" msgid="2588720613319969289">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‎‎‏‏‏‏‏‎‏‏‎‎‏‏‏‏‏‎‏‏‏‎‏‎‏‏‏‎‏‎‏‎‎‎‏‏‏‎‏‎‏‎‏‏‎‎‎‎‏‎‏‎‎‎‎‎‏‎‎‏‎Emergency alert: Extreme‎‏‎‎‏‎"</string>
-    <string name="cmas_extreme_immediate_observed_alert" msgid="2328845915287460780">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‎‎‎‎‎‏‎‏‎‎‎‏‏‎‏‏‏‎‎‏‎‎‎‎‎‏‎‏‎‏‏‎‏‏‎‏‎‏‎‏‎‏‎‎‎‎‎‏‏‎‏‏‏‎‏‎‏‏‎‎‎Emergency alert: Extreme‎‏‎‎‏‎"</string>
-    <string name="cmas_extreme_immediate_likely_alert" msgid="1859702950323471778">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‏‎‎‏‏‏‎‎‏‏‏‎‏‏‏‏‏‏‎‏‏‏‏‏‏‎‏‎‏‏‏‎‏‏‎‏‏‏‎‎‎‏‏‎‏‏‎‎‎‏‎‏‏‎‏‎‎‎‏‎‎Emergency alert: Extreme‎‏‎‎‏‎"</string>
-    <string name="cmas_severe_alert" msgid="4135809475315826913">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‎‏‎‏‏‎‎‏‎‏‎‏‎‏‏‎‎‎‏‏‎‎‏‏‏‎‎‎‏‎‏‎‏‎‏‏‏‎‎‏‎‏‎‎‎‎‎‎‎‎‏‏‏‎‎‎‎‏‎Emergency alert: Severe‎‏‎‎‏‎"</string>
-    <string name="cmas_amber_alert" msgid="6154867710264778887">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‏‎‏‎‏‏‎‏‎‏‎‎‏‏‏‏‎‏‏‏‎‎‏‎‎‏‎‏‎‎‏‎‏‎‏‏‏‎‏‏‎‏‎‏‎‏‎‎‎‎‎‏‎‎‎‎‏‏‏‎Child abduction (Amber alert)‎‏‎‎‏‎"</string>
-    <string name="cmas_required_monthly_test" msgid="1890205712251132193">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‏‎‏‎‎‎‏‏‏‎‏‏‎‏‎‏‏‏‎‎‎‎‎‏‎‏‎‎‏‎‎‏‏‏‏‏‎‎‏‎‎‏‎‎‏‎‎‏‏‎‎‏‎‎‏‎‎‎‎‏‎Required Monthly Test‎‏‎‎‏‎"</string>
-    <string name="cmas_exercise_alert" msgid="2892255514938370321">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‎‎‎‎‏‎‎‎‏‏‎‏‎‏‏‎‏‏‎‎‎‏‎‎‎‏‎‎‏‎‎‏‏‏‏‎‎‏‎‎‎‎‏‏‏‎‏‎‎‏‎‎‎‏‎‎‎‏‎Emergency alert (exercise)‎‏‎‎‏‎"</string>
-    <string name="cmas_operator_defined_alert" msgid="8755372450810011476">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‎‎‏‏‎‎‎‎‎‎‏‎‏‎‏‎‏‎‎‏‏‎‏‏‎‏‏‎‎‏‎‎‏‏‎‏‏‎‎‎‏‎‏‏‎‎‏‏‏‏‏‎‏‎‏‎‏‎‎‎Emergency alert (operator)‎‏‎‎‏‎"</string>
-    <string name="cb_other_message_identifiers" msgid="5790068194529377210">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‎‎‎‎‏‎‏‏‎‏‎‎‏‏‏‎‏‎‎‎‏‎‏‏‎‎‎‏‎‎‎‏‎‎‎‎‎‏‏‎‏‎‏‏‏‏‏‏‎‏‏‏‎‏‏‏‎‏‎‎Broadcast messages‎‏‎‎‏‎"</string>
-    <string name="public_safety_message" msgid="9119928798786998252">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‎‏‎‎‎‎‎‏‏‏‏‏‏‎‏‏‏‎‏‏‎‎‎‏‎‎‏‏‎‎‏‎‎‎‏‏‎‏‏‏‎‎‏‏‏‏‏‏‏‎‏‏‎‎‎Public safety message‎‏‎‎‏‎"</string>
-    <string name="state_local_test_alert" msgid="8003145745857480200">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‏‏‏‎‎‎‏‎‎‎‎‏‏‏‎‎‎‏‎‏‎‏‎‎‏‏‏‎‏‎‎‎‎‏‏‏‎‎‎‎‎‏‏‏‎‎‎‎‏‏‎‎‎‎‎‏‎‎‎‎State/Local test‎‏‎‎‏‎"</string>
-    <string name="emergency_alert" msgid="624783871477634263">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‎‏‎‏‎‏‎‏‏‏‎‏‎‏‏‎‏‏‎‎‏‏‏‏‎‏‏‏‏‎‎‎‎‏‏‎‏‎‎‎‏‎‏‎‏‎‏‎‎‏‏‎‏‎‏‏‏‎Emergency alert‎‏‎‎‏‎"</string>
-    <string name="emergency_alerts_title" msgid="6605036374197485429">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‏‏‏‎‏‎‏‎‎‏‏‏‎‎‏‏‎‏‏‎‎‏‎‏‏‎‎‏‎‏‏‏‎‎‏‎‎‏‎‎‏‏‎‎‏‏‏‏‏‏‎‏‏‏‎‏‎‏‎Alerts‎‏‎‎‏‎"</string>
-    <string name="notification_channel_broadcast_messages" msgid="880704362482824524">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‎‎‎‎‏‏‏‎‎‎‏‏‏‎‎‎‏‏‏‏‏‏‎‎‏‏‏‎‎‏‎‏‎‏‏‏‏‎‏‏‎‎‏‏‏‏‎‏‎‏‎‏‎‎‏‏‎‎‎Broadcast messages‎‏‎‎‏‎"</string>
-    <string name="notification_channel_emergency_alerts" msgid="5008287980979183617">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‎‏‏‎‎‎‎‎‎‏‎‎‎‎‎‎‏‏‎‏‏‎‎‎‏‎‎‎‏‏‎‏‏‏‎‎‎‏‏‏‏‏‎‏‏‎‎‎‎‎‎‎‎‎‎‎‎‏‎Emergency alerts‎‏‎‎‏‎"</string>
-    <string name="enable_alerts_master_toggle_title" msgid="1457904343636699446">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‎‎‎‏‏‏‎‏‏‏‎‎‎‎‏‎‎‎‏‎‎‎‏‏‎‏‎‏‏‎‏‏‎‏‏‏‏‎‏‏‎‏‏‏‎‏‎‎‏‎‎‏‏‎‏‏‎‎Allow alerts‎‏‎‎‏‎"</string>
-    <string name="enable_alerts_master_toggle_summary" msgid="699552922381916044">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‏‏‎‏‏‎‏‎‏‎‏‎‎‏‏‏‏‏‎‏‎‏‏‎‏‎‎‏‏‎‎‎‎‎‎‏‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‎‎‏‏‎‎‎Receive emergency notifications‎‏‎‎‏‎"</string>
-    <string name="alert_reminder_interval_title" msgid="7466642011937564868">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‏‏‏‎‎‏‏‏‏‎‏‏‎‏‎‏‏‏‎‏‎‏‏‎‎‏‏‎‎‏‎‎‏‎‏‎‎‎‏‎‏‏‏‏‏‎‏‎‎‎‏‏‎‎‎‏‎‎‎Alert reminder sound‎‏‎‎‏‎"</string>
-    <string name="enable_alert_speech_title" msgid="8052104771053526941">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‏‏‏‏‎‏‏‏‏‏‎‏‏‎‏‎‎‏‎‏‎‏‎‎‎‎‎‏‎‎‏‏‏‎‎‏‏‎‏‏‏‏‏‏‎‎‏‏‏‏‏‏‎‎‏‏‏‎‏‎Speak alert message‎‏‎‎‏‎"</string>
-    <string name="enable_alert_speech_summary" msgid="356086178962268831">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‏‏‎‎‏‏‏‏‎‎‎‏‎‎‎‏‎‎‏‎‎‏‏‏‏‏‏‎‎‎‏‏‏‏‏‎‎‏‏‏‏‎‎‎‏‏‏‎‎‎‏‎‏‎‎‏‏‏‏‏‎Use text-to-speech to speak emergency alert messages‎‏‎‎‏‎"</string>
-    <string name="emergency_alert_history_title" msgid="8310173569237268431">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‎‏‏‎‏‎‏‎‎‏‏‏‎‏‎‏‎‏‎‏‏‎‏‎‏‎‏‎‎‎‎‎‎‎‎‎‎‏‏‏‎‏‎‎‎‏‏‎‏‏‏‏‏‎‎‏‏‏‏‎Emergency alert history‎‏‎‎‏‎"</string>
-    <string name="alert_preferences_title" msgid="6001469026393248468">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‎‏‏‎‏‎‎‏‎‎‏‏‎‎‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‎‎‏‎‎‎‏‎‏‎‏‏‎‏‏‏‎‏‎‎‏‎‏‏‎‏‎‏‎‎‎Alert preferences‎‏‎‎‏‎"</string>
-    <string name="enable_etws_test_alerts_title" msgid="3593533226735441539">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‎‎‏‏‏‎‏‏‏‏‎‏‏‎‎‏‎‏‏‎‏‏‎‏‏‏‎‎‎‏‎‏‏‎‎‎‎‏‎‎‎‎‏‎‎‎‏‎‏‏‎‏‎‎‎‎‎‏‏‎ETWS test broadcasts‎‏‎‎‏‎"</string>
-    <string name="enable_etws_test_alerts_summary" msgid="8746155402612927306">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‎‎‏‎‏‏‎‎‎‎‎‏‎‎‏‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‏‎‎‎‏‏‎‏‏‎‏‏‎‏‎‏‏‎‏‎‎‏‎‏‎‎Test broadcasts for Earthquake Tsunami Warning System‎‏‎‎‏‎"</string>
-    <string name="enable_cmas_extreme_threat_alerts_title" msgid="5416260219062637770">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‎‏‏‎‎‏‎‏‎‏‎‎‏‏‎‏‎‏‏‏‏‏‏‏‎‎‏‎‎‎‎‎‎‎‎‎‏‎‎‏‏‏‎‏‏‎‏‎‏‎‎‏‏‎‎‏‎‏‎‎Extreme threats‎‏‎‎‏‎"</string>
-    <string name="enable_cmas_extreme_threat_alerts_summary" msgid="5832146246627518123">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‎‎‎‏‏‏‎‏‏‏‏‏‏‏‏‎‎‏‎‎‎‎‏‏‏‎‎‎‏‏‏‏‏‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‏‏‎‏‎‏‎‏‎‏‏‎Extreme threats to life and property‎‏‎‎‏‎"</string>
-    <string name="enable_cmas_severe_threat_alerts_title" msgid="1066172973703410042">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‏‎‏‏‎‎‏‎‏‏‏‏‎‎‏‏‏‎‏‎‏‎‏‎‏‏‏‎‏‏‎‏‏‎‏‎‏‏‎‏‏‎‏‎‏‏‎‎‎‏‎‏‏‏‏‎‏‎‎Severe threats‎‏‎‎‏‎"</string>
-    <string name="enable_cmas_severe_threat_alerts_summary" msgid="5292443310309039223">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‎‎‏‎‏‏‏‎‎‏‎‏‎‎‎‏‎‎‏‎‎‏‎‎‏‏‎‎‏‏‏‎‎‎‎‏‏‎‏‏‏‎‎‏‏‎‏‏‏‎‎‎‏‏‏‎‏‏‏‎Severe threats to life and property‎‏‎‎‏‎"</string>
-    <string name="enable_cmas_amber_alerts_title" msgid="1475030503498979651">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‎‎‏‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‏‏‎‏‏‎‎‎‏‎‎‏‏‏‏‎‏‏‎‎‎‎‎‏‏‎‎‏‎‏‎‎‎‎‏‏‎AMBER alerts‎‏‎‎‏‎"</string>
-    <string name="enable_cmas_amber_alerts_summary" msgid="4495233280416889667">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‏‏‎‎‏‏‎‎‎‏‎‎‏‎‎‎‏‏‎‏‏‎‏‏‎‏‎‎‎‎‎‏‏‎‏‏‏‎‎‎‎‏‏‎‎‏‏‏‎‏‏‎‏‎‎‎‎‏‏‎Child abduction emergency bulletins‎‏‎‎‏‎"</string>
-    <string name="enable_alert_message_title" msgid="2939830587633599352">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‎‎‏‏‎‎‏‏‎‎‎‏‏‎‎‎‎‎‎‏‎‏‎‏‏‏‏‏‏‏‎‎‏‏‏‎‏‏‏‎‏‎‏‎‎‎‎‎‏‏‎‏‏‏‏‎‎‎‎Alert messages‎‏‎‎‏‎"</string>
-    <string name="enable_alert_message_summary" msgid="6525664541696985610">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‏‎‏‎‎‎‏‏‏‏‏‏‎‏‎‎‎‏‎‏‎‏‎‎‏‏‏‎‎‎‎‎‏‏‎‎‏‏‎‏‏‎‎‎‎‎‏‎‏‎‎‎‎‎‏‎‏‎‎Warn about imminent safety threats‎‏‎‎‏‎"</string>
-    <string name="enable_public_safety_messages_title" msgid="5576770949182656524">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‏‎‏‎‏‏‎‎‏‎‎‏‎‏‎‏‎‏‏‏‎‏‎‎‎‎‏‏‎‎‏‏‎‏‏‏‏‎‎‎‎‎‎‎‎‎‏‎‎‎‎‎‎‎‎‏‏‎‎‎Public safety messages‎‏‎‎‏‎"</string>
-    <string name="enable_public_safety_messages_summary" msgid="7868069748857851521">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‏‎‏‎‎‏‏‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‏‏‏‏‏‏‏‏‏‎‎‎‎‏‎‏‎‏‎‏‏‎‏‎‎‏‏‏‎‏‎‎‎‎‎‎‏‎Recommended actions that can save lives or property‎‏‎‎‏‎"</string>
-    <string name="enable_state_local_test_alerts_title" msgid="1012930918171302720">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‏‎‎‎‎‎‏‏‏‎‏‎‏‎‎‏‏‏‎‏‎‎‏‏‎‏‏‎‏‎‎‏‏‏‏‎‎‏‏‏‎‎‎‏‎‏‏‎‏‏‎‏‎‎‎‎‎‎‎State and local tests‎‏‎‎‏‎"</string>
-    <string name="enable_state_local_test_alerts_summary" msgid="780298327377950187">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‏‎‏‏‎‏‎‏‎‎‎‎‏‎‏‏‎‏‎‎‏‏‎‎‎‏‏‏‎‎‏‏‏‎‎‎‏‏‎‎‎‏‏‎‏‏‎‏‎‏‏‏‏‎‏‎‏‏‎Receive test messages from state and local authorities‎‏‎‎‏‎"</string>
-    <string name="enable_emergency_alerts_message_title" msgid="661894007489847468">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‏‎‎‏‎‏‏‏‏‏‎‎‎‎‏‎‏‎‎‎‏‎‏‏‏‏‏‎‎‏‎‏‏‎‏‏‎‎‏‎‎‎‏‏‎‎‎‎‎‏‎‏‎‏‏‎‎‎Emergency alerts‎‏‎‎‏‎"</string>
-    <string name="enable_emergency_alerts_message_summary" msgid="7574617515441602546">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‎‎‏‎‎‎‏‏‏‏‎‎‏‏‏‎‎‏‎‎‏‏‏‏‏‏‏‏‎‎‎‏‏‎‏‎‏‎‏‏‎‎‏‎‎‏‏‏‏‏‏‏‏‏‏‎‎‏‎‎Warn about life-threatening events‎‏‎‎‏‎"</string>
-    <string name="enable_cmas_test_alerts_title" msgid="1319231576046146463">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‎‏‎‎‏‎‎‏‏‏‎‏‏‎‏‏‎‏‎‎‎‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‏‏‎‏‎‏‏‎‎‏‎‎‏‏‏‎‎‏‏‏‏‏‎Required monthly test‎‏‎‎‏‎"</string>
-    <string name="enable_cmas_test_alerts_summary" msgid="588517403520088484">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‎‎‎‏‎‏‎‏‎‏‏‎‏‎‏‎‏‎‏‏‏‎‏‎‏‏‏‎‏‏‏‎‎‏‏‏‏‎‎‏‏‎‎‏‏‏‎‎‏‏‎‏‎‎‏‎‎‎Receive test messages for the safety alert system‎‏‎‎‏‎"</string>
-    <string name="enable_alert_vibrate_title" msgid="982817538757982090">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‎‏‏‎‏‎‎‎‏‏‏‎‏‎‏‎‏‏‎‏‎‏‏‎‎‎‎‎‏‎‎‏‎‎‏‎‏‏‎‎‎‏‎‏‏‏‎‏‏‏‏‎‎‎‏‎‏‎‎Vibrate‎‏‎‎‏‎"</string>
-    <string name="use_full_volume_title" msgid="8581439612945182255">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‏‏‎‎‎‏‎‏‏‏‎‏‏‎‎‏‎‏‏‏‎‏‏‏‎‎‏‎‎‎‏‏‏‏‎‎‎‏‎‎‎‎‎‎‏‎‏‏‏‎‎‎‏‎‏‏‏‏‎Use full volume‎‏‎‎‏‎"</string>
-    <string name="use_full_volume_summary" msgid="3560764277281704697">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‎‎‏‎‏‏‎‏‎‏‎‎‏‏‎‎‎‎‎‎‎‏‏‏‏‏‎‏‎‎‏‎‏‎‎‎‏‎‎‎‏‏‎‏‏‎‎‏‎‏‎‏‏‏‏‏‎‎‏‎Ignore other volume and Do Not Disturb preferences‎‏‎‎‏‎"</string>
-    <string name="enable_area_update_info_alerts_title" msgid="3442042268424617226">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‏‏‏‏‏‎‎‎‏‎‎‏‎‎‏‎‏‏‏‎‎‏‏‎‏‎‎‏‏‎‏‎‏‏‏‎‏‎‏‎‏‎‎‎‎‏‏‏‏‎‏‎‎‎‎‏‎‏‎‎Area update broadcasts‎‏‎‎‏‎"</string>
-    <string name="enable_area_update_info_alerts_summary" msgid="6437816607144264910">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‎‏‎‏‎‏‎‏‏‏‏‎‏‏‏‎‎‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‎‏‏‏‏‏‎‎‎‏‏‏‎‎‎‏‎‎‏‏‎‎‏‏‏‎‎Show update information in SIM status‎‏‎‎‏‎"</string>
-    <string name="category_dev_settings_title" msgid="6194393458398329994">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‏‎‏‏‏‏‏‎‏‏‎‏‏‏‎‏‎‎‎‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‏‏‎‏‎‏‎‏‏‏‏‏‏‏‎‎‏‎‎‎‏‎‏‎‎Developer options‎‏‎‎‏‎"</string>
-    <string name="cmas_category_heading" msgid="3923503130776640717">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‏‏‎‎‏‏‏‎‎‏‏‎‎‎‏‎‏‎‏‎‏‎‎‏‏‏‏‎‏‎‏‎‎‎‏‎‎‎‏‎‎‏‏‎‎‎‏‏‏‎‎‏‏‎‎‏‏‎‏‎Alert Category:‎‏‎‎‏‎"</string>
-    <string name="cmas_category_geo" msgid="4979494217069688527">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‎‏‎‎‎‏‏‎‏‎‏‎‏‏‎‏‏‏‏‎‎‏‏‎‏‏‏‎‏‎‎‎‎‏‏‎‎‎‏‏‎‎‏‏‏‏‎‏‏‎‏‏‎‎‏‏‏‏‎Geophysical‎‏‎‎‏‎"</string>
-    <string name="cmas_category_met" msgid="7563732573851773537">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‎‎‎‏‏‏‏‎‏‏‏‏‏‎‎‎‏‏‎‏‎‏‏‎‎‏‏‏‎‎‏‏‎‎‎‎‏‏‏‏‏‎‎‎‏‏‎‏‎‏‎‎‏‏‎‎‎‎‏‎Meteorological‎‏‎‎‏‎"</string>
-    <string name="cmas_category_safety" msgid="2986472639641883453">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‎‏‎‏‏‏‎‎‏‎‎‎‎‏‎‏‎‏‎‎‎‎‏‎‏‎‏‏‏‏‎‎‎‏‎‏‎‎‎‏‏‏‎‎‎‏‎‎‏‏‎‎‏‏‏‏‎‏‎Safety‎‏‎‎‏‎"</string>
-    <string name="cmas_category_security" msgid="2549520159044403704">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‎‎‏‏‎‏‏‎‎‎‎‏‏‎‏‏‎‏‏‏‎‎‎‏‎‎‏‏‏‎‏‎‎‎‏‏‏‎‎‎‏‎‏‎‎‎‎‎‎‎‎‏‏‏‏‏‏‎‎‎‎Security‎‏‎‎‏‎"</string>
-    <string name="cmas_category_rescue" msgid="4907571719983321086">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‎‎‎‎‎‏‏‎‏‏‎‎‏‏‎‎‏‎‎‏‏‏‏‎‏‎‎‏‎‏‎‏‏‏‎‏‎‏‏‎‏‏‏‏‎‎‏‏‏‏‏‏‏‏‏‏‏‎‎Rescue‎‏‎‎‏‎"</string>
-    <string name="cmas_category_fire" msgid="3331981591918341119">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‏‏‎‎‎‏‏‏‏‎‏‏‎‎‏‎‎‏‏‏‎‎‏‏‎‏‎‏‎‏‏‎‎‎‎‎‏‏‎‏‏‏‎‏‏‏‎‏‎‏‏‏‏‏‏‏‏‏‏‎Fire‎‏‎‎‏‎"</string>
-    <string name="cmas_category_health" msgid="312569774587117324">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‏‏‎‎‎‏‎‏‎‏‏‎‎‏‏‏‏‎‎‎‏‎‎‎‏‏‎‏‏‏‎‎‏‏‎‏‏‏‎‎‏‎‎‏‏‏‏‏‏‏‏‏‎‎‎‎‏‏‎‎‎Health‎‏‎‎‏‎"</string>
-    <string name="cmas_category_env" msgid="9213088574227522961">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‏‏‏‏‎‏‏‎‏‏‎‏‏‏‎‏‏‏‎‎‏‏‏‏‏‎‏‏‎‎‎‏‏‏‎‎‎‎‏‎‏‎‏‎‏‎‎‏‎‏‏‎‎‏‎‎‎‏‎Environmental‎‏‎‎‏‎"</string>
-    <string name="cmas_category_transport" msgid="5014901635987361642">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‎‏‏‎‎‏‏‎‎‎‏‎‎‎‎‎‏‎‎‏‏‏‎‏‏‏‏‏‎‎‎‏‎‎‏‎‎‎‏‏‎‏‎‏‎‎‏‏‏‏‎‏‏‎‏‎‏‎‎Transportation‎‏‎‎‏‎"</string>
-    <string name="cmas_category_infra" msgid="3558151044446851398">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‎‎‏‎‏‏‎‎‎‎‏‎‎‎‏‎‏‏‏‏‎‎‎‎‏‎‏‏‏‏‎‎‏‎‏‎‏‏‏‎‎‎‎‏‎‎‏‎‎‎‏‎‏‎‎‎‏‏‎‎Infrastructure‎‏‎‎‏‎"</string>
-    <string name="cmas_category_cbrne" msgid="240421557913603971">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‎‏‏‎‏‎‏‎‏‏‎‎‎‏‎‎‏‏‎‎‎‏‎‎‎‏‎‎‎‎‏‎‎‏‏‏‏‎‎‏‎‎‏‎‏‎‎‏‏‏‏‏‎‎‎‎‎‏‏‎Chemical/Biological/Nuclear/Explosive‎‏‎‎‏‎"</string>
-    <string name="cmas_category_other" msgid="6158932360790744360">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‏‎‏‎‏‏‏‏‎‎‎‏‏‏‎‏‏‎‎‎‏‎‏‏‎‎‏‏‏‎‎‏‎‎‎‏‎‎‏‎‏‏‏‎‎‎‏‏‏‎‏‎‎‏‎‏‎‎‎‎Other‎‏‎‎‏‎"</string>
-    <string name="cmas_response_heading" msgid="4205379547245540163">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‏‎‎‏‎‏‏‏‎‎‏‎‎‎‎‎‏‎‎‏‏‎‏‎‏‎‏‏‎‎‏‎‏‏‎‏‎‎‎‎‏‏‎‏‏‏‎‎‏‏‎‏‎‎‎‎‏‏‎Response Type:‎‏‎‎‏‎"</string>
-    <string name="cmas_response_shelter" msgid="7301175579079615909">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‎‏‎‏‎‏‎‎‏‎‏‏‏‏‏‏‎‎‏‎‎‎‎‎‎‏‏‎‎‎‎‎‎‏‏‏‎‎‏‏‎‎‎‎‏‎‎‏‎‏‏‎‏‎‎‏‎‏‎Shelter‎‏‎‎‏‎"</string>
-    <string name="cmas_response_evacuate" msgid="5833170084430021095">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‎‎‎‏‏‏‏‎‎‏‏‏‎‎‏‎‏‎‏‎‏‎‎‏‎‎‏‎‏‎‎‏‏‎‎‏‏‏‎‎‎‏‏‎‎‎‎‎‎‎‏‏‏‏‎‎‏‏‏‎Evacuate‎‏‎‎‏‎"</string>
-    <string name="cmas_response_prepare" msgid="8428073909753758319">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‎‎‏‏‏‏‎‏‏‎‏‎‎‎‏‎‎‎‏‎‎‏‎‎‎‏‎‎‏‎‏‎‏‏‏‎‎‏‏‏‎‎‎‏‎‎‎‎‏‎‎‏‏‎‏‏‏‏‎Prepare‎‏‎‎‏‎"</string>
-    <string name="cmas_response_execute" msgid="284719420769568493">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‎‏‏‏‏‏‏‎‎‏‏‏‎‎‎‎‏‏‎‏‏‎‎‏‏‎‏‏‏‏‏‏‏‏‏‏‎‏‎‎‎‎‎‎‎‎‎‎‏‏‎‏‏‏‎‏‏‎‏‎Execute‎‏‎‎‏‎"</string>
-    <string name="cmas_response_monitor" msgid="681400164440495749">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‏‎‏‏‏‎‏‎‎‏‏‎‏‎‎‎‏‏‏‎‏‎‏‏‎‏‏‎‏‏‏‏‏‎‏‎‏‏‏‏‎‎‏‎‏‎‏‏‎‏‎‎‎‎‏‎‏‎Monitor‎‏‎‎‏‎"</string>
-    <string name="cmas_response_avoid" msgid="156419597612629270">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‎‏‎‎‎‏‎‏‎‏‏‏‎‏‏‎‏‏‎‏‏‎‎‏‎‏‏‏‎‎‏‎‏‏‏‏‎‎‏‏‏‎‎‎‎‏‏‏‏‎‏‎‎‎‏‎‏‏‎‎Avoid‎‏‎‎‏‎"</string>
-    <string name="cmas_response_assess" msgid="9043534222710563415">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‎‏‏‎‎‎‎‎‎‏‎‎‎‏‎‏‏‎‎‏‏‏‎‏‎‏‏‏‏‏‏‎‏‎‏‏‎‎‎‎‎‎‎‏‏‎‎‎‏‎‎‏‎‏‎‏‏‏‎Assess‎‏‎‎‏‎"</string>
-    <string name="cmas_response_none" msgid="5149009359674452959">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‏‏‎‏‏‏‎‏‎‎‏‏‏‏‎‏‎‎‏‎‏‏‏‎‏‏‎‏‏‎‎‏‎‎‏‎‎‎‏‏‏‎‏‎‏‎‏‎‏‏‏‏‎‏‏‏‏‏‎None‎‏‎‎‏‎"</string>
-    <string name="cmas_severity_heading" msgid="8437057117822305243">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‎‏‎‎‎‏‎‏‏‎‎‏‏‏‎‎‏‎‏‎‏‏‏‏‏‏‎‏‎‎‏‎‎‏‎‏‏‏‏‎‏‎‏‏‎‎‎‎‏‏‏‏‎‏‏‎‏‏‎Severity:‎‏‎‎‏‎"</string>
-    <string name="cmas_severity_extreme" msgid="1312013282860183082">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‎‏‎‎‎‏‏‎‏‎‏‎‎‏‏‎‏‎‏‎‎‏‎‎‎‎‏‎‎‏‎‎‎‎‏‎‏‏‏‏‎‎‏‏‎‏‎‏‏‏‎‎‎‏‎‏‎‏‎‎Extreme‎‏‎‎‏‎"</string>
-    <string name="cmas_severity_severe" msgid="7504359209737074524">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‎‎‎‎‎‏‎‎‏‎‎‏‏‎‏‎‏‏‎‏‏‏‏‎‎‎‏‎‎‎‎‎‎‏‏‎‏‎‎‏‏‏‏‎‎‏‏‎‏‏‏‎‏‎‏‏‏‎‎‎Severe‎‏‎‎‏‎"</string>
-    <string name="cmas_urgency_heading" msgid="8218282767913431492">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‎‏‎‎‎‎‎‏‏‎‏‎‎‏‏‎‏‎‎‏‎‏‎‎‎‏‎‏‎‎‏‎‏‎‏‎‎‏‏‎‎‎‏‏‎‏‏‎‎‎‏‏‏‎‎‎‏‎‎‎Urgency:‎‏‎‎‏‎"</string>
-    <string name="cmas_urgency_immediate" msgid="1577485208196449288">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‏‏‏‏‎‎‏‎‎‎‏‎‏‏‎‏‎‎‏‏‏‎‎‎‎‎‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‎‎‎‏‎‎‎‎‎‎‏‎‎‎‎Immediate‎‏‎‎‏‎"</string>
-    <string name="cmas_urgency_expected" msgid="6830831119872375936">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‏‏‎‏‏‎‎‏‎‏‏‏‏‏‏‏‏‎‎‏‎‏‏‎‏‏‎‎‏‎‏‎‏‎‎‏‎‏‎‏‏‎‏‎‎‏‏‏‏‎‎‏‎‎‎‎‎‎‎‎Expected‎‏‎‎‏‎"</string>
-    <string name="cmas_certainty_heading" msgid="8374669249736439193">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‎‎‎‎‏‏‏‎‎‎‏‏‎‎‏‏‎‏‎‏‎‎‏‏‏‏‏‏‎‎‏‏‎‏‏‏‏‎‏‏‎‏‎‏‏‏‎‏‎‏‏‎‎‏‏‎‎‏‎Certainty:‎‏‎‎‏‎"</string>
-    <string name="cmas_certainty_observed" msgid="3668549749352106472">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‎‏‎‏‏‏‎‏‎‎‏‎‏‎‎‏‏‏‎‏‎‎‎‏‏‏‏‎‏‏‎‏‎‏‎‏‏‏‏‏‎‏‏‎‎‎‎‏‎‎‏‏‏‏‎‏‎‎‎‎Observed‎‏‎‎‏‎"</string>
-    <string name="cmas_certainty_likely" msgid="4254497828943291749">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‏‏‎‎‎‎‏‎‏‏‎‎‎‎‎‎‏‏‎‎‏‏‏‎‏‏‏‏‏‎‏‏‏‏‏‏‎‏‎‏‎‏‏‏‏‎‎‏‎‏‎‏‏‎‎‏‎‏‎Likely‎‏‎‎‏‎"</string>
-    <string name="delivery_time_heading" msgid="5980836543433619329">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‎‏‏‎‎‎‎‎‎‎‎‎‎‏‏‎‎‏‏‎‎‏‎‎‏‎‏‏‏‏‏‏‏‏‏‎‎‎‎‏‎‎‏‎‎‏‏‎‏‏‏‏‎‎‎‎‎‎‏‎Received:‎‏‎‎‏‎"</string>
-    <string name="notification_multiple" msgid="5121978148152124860">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‏‏‎‎‎‏‎‏‎‎‏‏‏‎‏‎‏‏‏‏‏‏‏‏‎‏‎‎‎‎‎‎‏‏‏‏‏‎‏‎‏‏‏‎‏‎‎‏‎‏‏‎‏‏‏‏‎‎‎‎‏‎‎‏‏‎<xliff:g id="COUNT">%s</xliff:g>‎‏‎‎‏‏‏‎ unread alerts.‎‏‎‎‏‎"</string>
-    <string name="notification_multiple_title" msgid="1523638925739947855">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‏‎‎‏‎‎‏‎‏‎‎‎‎‏‏‎‏‏‎‎‎‎‏‏‏‏‏‎‎‎‏‏‎‏‎‏‎‏‏‏‏‏‏‏‏‎‏‏‏‎‏‎‎‏‏‏‏‎New alerts‎‏‎‎‏‎"</string>
-    <string name="show_cmas_opt_out_summary" msgid="6926059266585295440">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‎‎‎‎‎‎‏‏‏‏‎‎‏‎‎‏‏‏‎‎‎‏‏‎‏‎‏‎‎‎‎‎‏‏‎‎‎‎‏‏‏‏‎‎‏‏‏‏‎‏‎‎‏‎‏‎‎‎‎‎Show an opt-out dialog after displaying the first alert (other than Presidential Alert).‎‏‎‎‏‎"</string>
-    <string name="show_cmas_opt_out_title" msgid="9182104842820171132">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‏‏‎‏‏‎‏‏‎‏‎‏‏‎‎‎‏‏‏‎‏‏‎‏‎‎‏‎‏‎‎‏‏‎‏‎‏‏‎‎‎‎‏‎‏‏‎‏‎‏‎‏‏‏‏‏‎‎‎Show opt-out dialog‎‏‎‎‏‎"</string>
-    <string name="cmas_opt_out_dialog_text" msgid="7529010670998259128">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‎‎‎‎‏‏‏‏‏‎‎‎‏‏‎‏‎‏‏‎‏‎‏‎‎‎‎‏‎‎‏‏‎‏‎‏‏‏‏‏‎‏‎‎‎‎‎‎‏‎‏‏‎‏‏‏‎‎‎‎You are currently receiving Emergency Alerts. Would you like to continue receiving Emergency Alerts?‎‏‎‎‏‎"</string>
-    <string name="cmas_opt_out_button_yes" msgid="7248930667195432936">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‎‎‏‎‎‏‏‎‎‏‎‏‏‎‎‎‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‎‏‏‏‎‎‏‏‏‏‎‎‏‏‏‏‏‏‏‏‏‏‎‏‎‎‎‎Yes‎‏‎‎‏‎"</string>
-    <string name="cmas_opt_out_button_no" msgid="3110484064328538553">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‏‏‎‎‏‎‏‎‏‎‏‎‏‎‏‎‎‎‏‏‎‎‎‏‏‎‏‏‎‎‎‏‎‏‏‏‎‏‎‎‎‎‏‏‎‏‏‎‎‏‏‎‏‏‏‎‎‏‎No‎‏‎‎‏‎"</string>
-    <string name="cb_list_activity_title" msgid="1433502151877791724">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‎‏‏‏‏‏‎‎‏‎‎‏‏‎‏‎‎‏‎‏‎‎‏‏‏‎‎‏‎‏‏‎‎‏‏‎‏‏‎‏‏‏‏‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‎‎Emergency alert history‎‏‎‎‏‎"</string>
+    <string name="app_label" msgid="7559008249836700599">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‎‎‎‏‏‏‎‎‏‏‎‏‏‏‏‏‏‎‏‏‏‏‏‎‏‎‎‎‎‏‏‎‎‎‏‏‏‎‎‎‎‏‏‏‎‎‎‎‎‏‏‏‎‏‏‎‏‏‏‎Cell Broadcasts‎‏‎‎‏‎"</string>
+    <string name="sms_cb_settings" msgid="3795852619005211509">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‏‎‎‏‎‏‎‏‏‎‏‏‎‎‏‎‎‏‏‏‏‎‏‏‎‎‎‎‏‏‏‏‏‏‏‏‎‎‎‏‎‏‎‏‎‏‏‎‎‏‏‎‏‏‏‎‏‎‏‎Emergency alerts‎‏‎‎‏‎"</string>
+    <string name="cell_broadcast_settings_not_available" msgid="2612364350745967915">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‎‏‎‎‎‏‎‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‎‎‏‎‎‏‎‎‏‏‏‏‏‎‎‎‎‏‏‎‏‏‎‏‏‏‏‏‎‏‎‎‏‎‏‎‏‏‎Cell Broadcast settings are not available for this user‎‏‎‎‏‎"</string>
+    <string name="button_dismiss" msgid="1234221657930516287">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‎‎‏‎‎‏‎‎‎‎‎‏‏‎‏‎‏‏‎‎‎‎‎‏‏‏‏‎‎‎‎‎‏‏‏‎‎‎‎‏‎‏‎‏‏‎‎‎‏‏‏‎‎‏‏‏‏‏‏‎OK‎‏‎‎‏‎"</string>
+    <string name="no_cell_broadcasts" msgid="5409324337492758562">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‎‏‏‎‎‎‏‎‎‎‏‏‏‎‎‎‏‏‏‏‏‎‏‎‎‏‏‎‎‏‏‏‎‏‏‎‎‏‎‎‎‎‏‏‎‎‏‎‎‎‎‎‎‏‎‎‎‏‎‎There are no active alerts in your area. You can change the Alert settings using the Settings menu option.‎‏‎‎‏‎"</string>
+    <string name="menu_delete_all" msgid="3940997343921149800">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‏‏‎‏‎‏‏‎‎‎‏‎‎‏‏‏‏‎‎‎‎‏‏‎‏‎‎‎‎‏‏‏‎‎‏‎‎‏‎‏‏‎‏‏‎‎‏‎‏‏‏‎‏‏‎‏‎‎‎‎Delete broadcasts‎‏‎‎‏‎"</string>
+    <string name="message_options" msgid="3178489901903589574">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‏‎‎‎‎‎‏‏‏‎‎‎‏‎‎‎‎‏‏‏‎‏‏‏‎‎‏‏‎‏‏‎‏‎‏‎‎‏‎‎‎‎‏‎‎‏‏‎‎‎‎‏‏‎‎‎‏‏‎‎Message options‎‏‎‎‏‎"</string>
+    <string name="menu_view_details" msgid="1040989019045280975">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‏‎‎‏‏‏‎‎‏‎‎‏‎‏‎‏‎‏‏‏‏‏‏‏‏‏‎‏‎‏‎‏‏‏‎‏‎‎‎‏‏‎‏‎‏‏‏‏‎‎‏‏‎‎‏‏‏‏‎View details‎‏‎‎‏‎"</string>
+    <string name="menu_delete" msgid="128380070910799366">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‎‏‏‏‏‎‎‏‎‎‎‎‎‎‏‏‎‎‎‏‏‏‏‏‏‏‎‏‎‎‎‏‎‏‎‏‏‏‎‎‎‎‎‏‎‏‎‏‏‏‎‎‎‎‎‎‏‏‎‎Delete broadcast‎‏‎‎‏‎"</string>
+    <string name="view_details_title" msgid="1780427629491781473">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‏‎‎‎‏‎‏‏‎‏‎‏‎‏‎‏‏‎‎‏‎‏‏‏‏‏‏‎‏‏‏‏‎‏‎‎‎‎‏‎‎‏‎‏‏‏‏‏‏‏‏‏‎‏‏‎‎‎‎‏‎Alert details‎‏‎‎‏‎"</string>
+    <string name="confirm_delete_broadcast" msgid="2540199303730232322">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‎‎‏‏‎‏‎‎‎‎‎‎‏‎‎‏‏‎‎‏‏‏‎‎‏‏‏‎‏‏‏‎‎‏‎‏‏‎‎‎‏‎‎‎‏‏‏‏‏‏‎‎‎‎‎‎‎‎‏‎‎Delete this broadcast?‎‏‎‎‏‎"</string>
+    <string name="confirm_delete_all_broadcasts" msgid="2924444089047280871">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‎‎‏‎‎‏‎‏‎‏‏‎‏‏‎‏‏‎‎‏‏‎‎‏‏‏‏‎‎‏‎‏‎‏‎‎‏‎‎‏‏‏‎‏‏‎‏‏‎‎‏‏‏‎‎‏‏‏‎Delete all received broadcast messages?‎‏‎‎‏‎"</string>
+    <string name="button_delete" msgid="4672451757925194350">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‎‎‎‏‏‎‏‎‏‏‏‏‏‏‎‎‎‏‎‎‎‎‏‏‏‏‎‎‎‏‏‏‏‎‏‎‏‎‎‎‏‏‏‏‏‎‏‏‎‏‎‎‏‏‎‏‏‏‎‎Delete‎‏‎‎‏‎"</string>
+    <string name="button_cancel" msgid="7479958360523246140">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‏‏‏‏‎‎‏‏‏‎‎‎‏‎‎‏‏‎‎‏‏‏‏‏‏‏‏‎‎‏‎‏‎‏‏‎‏‏‎‎‏‏‏‎‎‎‏‎‏‎‎‎‏‏‏‏‎‎‎Cancel‎‏‎‎‏‎"</string>
+    <string name="etws_earthquake_warning" msgid="6428741104423152511">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‎‏‎‎‏‏‎‏‏‏‎‏‏‏‏‎‎‏‏‏‏‏‏‎‎‏‎‎‏‏‏‎‏‎‎‎‎‏‏‎‎‎‎‎‏‎‏‎‏‏‎‏‏‏‏‏‏‏‎Earthquake warning‎‏‎‎‏‎"</string>
+    <string name="etws_tsunami_warning" msgid="6173964105145900312">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‏‎‏‏‎‏‎‏‏‏‎‎‏‎‏‎‎‏‏‏‎‏‎‎‏‎‎‎‏‏‏‏‏‏‏‎‎‏‏‏‎‏‏‏‏‎‏‏‏‎‏‎‎‎‏‏‎‎‎‎Tsunami warning‎‏‎‎‏‎"</string>
+    <string name="etws_earthquake_and_tsunami_warning" msgid="662449983177407681">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‏‎‎‏‏‎‎‎‏‎‏‏‏‏‏‏‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‏‎‎‎‎‎‎‏‎‏‏‏‏‎‎‏‏‎‎‎‎‎‏‎Earthquake and tsunami warning‎‏‎‎‏‎"</string>
+    <string name="etws_test_message" msgid="8447820262584381894">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‎‏‎‎‏‏‏‏‎‎‏‎‏‎‏‏‏‏‏‏‎‎‎‏‎‏‎‎‏‏‎‏‎‏‏‎‏‏‎‎‎‎‎‏‏‏‎‎‎‏‏‏‎‎‎‏‏‎‎ETWS test message‎‏‎‎‏‎"</string>
+    <string name="etws_other_emergency_type" msgid="5233080551309721499">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‎‎‎‏‎‎‏‏‏‏‏‏‎‏‎‎‎‏‏‎‎‎‎‏‎‎‏‎‎‎‎‏‏‎‏‏‏‎‏‎‎‏‎‎‎‏‏‏‏‏‏‏‎‎‏‏‎‏‏‎Emergency warning‎‏‎‎‏‎"</string>
+    <string name="cmas_presidential_level_alert" msgid="1209234030582361001">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‎‎‎‏‏‎‎‏‎‎‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‎‏‏‎‏‎‏‎‏‏‎‎‏‎‏‏‎‎‏‏‏‎‏‎‏‏‏‎‏‎‏‎‎‏‎Presidential alert‎‏‎‎‏‎"</string>
+    <string name="cmas_extreme_alert" msgid="2588720613319969289">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‎‎‏‏‏‏‏‎‏‏‎‎‏‏‏‏‏‎‏‏‏‎‏‎‏‏‏‎‏‎‏‎‎‎‏‏‏‎‏‎‏‎‏‏‎‎‎‎‏‎‏‎‎‎‎‎‏‎‎‏‎Emergency alert: Extreme‎‏‎‎‏‎"</string>
+    <string name="cmas_extreme_immediate_observed_alert" msgid="2328845915287460780">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‎‎‎‎‎‏‎‏‎‎‎‏‏‎‏‏‏‎‎‏‎‎‎‎‎‏‎‏‎‏‏‎‏‏‎‏‎‏‎‏‎‏‎‎‎‎‎‏‏‎‏‏‏‎‏‎‏‏‎‎‎Emergency alert: Extreme‎‏‎‎‏‎"</string>
+    <string name="cmas_extreme_immediate_likely_alert" msgid="1859702950323471778">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‏‎‎‏‏‏‎‎‏‏‏‎‏‏‏‏‏‏‎‏‏‏‏‏‏‎‏‎‏‏‏‎‏‏‎‏‏‏‎‎‎‏‏‎‏‏‎‎‎‏‎‏‏‎‏‎‎‎‏‎‎Emergency alert: Extreme‎‏‎‎‏‎"</string>
+    <string name="cmas_severe_alert" msgid="4135809475315826913">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‎‏‎‏‏‎‎‏‎‏‎‏‎‏‏‎‎‎‏‏‎‎‏‏‏‎‎‎‏‎‏‎‏‎‏‏‏‎‎‏‎‏‎‎‎‎‎‎‎‎‏‏‏‎‎‎‎‏‎Emergency alert: Severe‎‏‎‎‏‎"</string>
+    <string name="cmas_amber_alert" msgid="6154867710264778887">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‏‎‏‎‏‏‎‏‎‏‎‎‏‏‏‏‎‏‏‏‎‎‏‎‎‏‎‏‎‎‏‎‏‎‏‏‏‎‏‏‎‏‎‏‎‏‎‎‎‎‎‏‎‎‎‎‏‏‏‎Child abduction (Amber alert)‎‏‎‎‏‎"</string>
+    <string name="cmas_required_monthly_test" msgid="1890205712251132193">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‏‎‏‎‎‎‏‏‏‎‏‏‎‏‎‏‏‏‎‎‎‎‎‏‎‏‎‎‏‎‎‏‏‏‏‏‎‎‏‎‎‏‎‎‏‎‎‏‏‎‎‏‎‎‏‎‎‎‎‏‎Required Monthly Test‎‏‎‎‏‎"</string>
+    <string name="cmas_exercise_alert" msgid="2892255514938370321">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‎‎‎‎‏‎‎‎‏‏‎‏‎‏‏‎‏‏‎‎‎‏‎‎‎‏‎‎‏‎‎‏‏‏‏‎‎‏‎‎‎‎‏‏‏‎‏‎‎‏‎‎‎‏‎‎‎‏‎Emergency alert (exercise)‎‏‎‎‏‎"</string>
+    <string name="cmas_operator_defined_alert" msgid="8755372450810011476">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‎‎‏‏‎‎‎‎‎‎‏‎‏‎‏‎‏‎‎‏‏‎‏‏‎‏‏‎‎‏‎‎‏‏‎‏‏‎‎‎‏‎‏‏‎‎‏‏‏‏‏‎‏‎‏‎‏‎‎‎Emergency alert (operator)‎‏‎‎‏‎"</string>
+    <string name="cb_other_message_identifiers" msgid="5790068194529377210">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‎‎‎‎‏‎‏‏‎‏‎‎‏‏‏‎‏‎‎‎‏‎‏‏‎‎‎‏‎‎‎‏‎‎‎‎‎‏‏‎‏‎‏‏‏‏‏‏‎‏‏‏‎‏‏‏‎‏‎‎Broadcast messages‎‏‎‎‏‎"</string>
+    <string name="public_safety_message" msgid="9119928798786998252">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‎‏‎‎‎‎‎‏‏‏‏‏‏‎‏‏‏‎‏‏‎‎‎‏‎‎‏‏‎‎‏‎‎‎‏‏‎‏‏‏‎‎‏‏‏‏‏‏‏‎‏‏‎‎‎Public safety message‎‏‎‎‏‎"</string>
+    <string name="state_local_test_alert" msgid="8003145745857480200">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‏‏‏‎‎‎‏‎‎‎‎‏‏‏‎‎‎‏‎‏‎‏‎‎‏‏‏‎‏‎‎‎‎‏‏‏‎‎‎‎‎‏‏‏‎‎‎‎‏‏‎‎‎‎‎‏‎‎‎‎State/Local test‎‏‎‎‏‎"</string>
+    <string name="emergency_alert" msgid="624783871477634263">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‎‏‎‏‎‏‎‏‏‏‎‏‎‏‏‎‏‏‎‎‏‏‏‏‎‏‏‏‏‎‎‎‎‏‏‎‏‎‎‎‏‎‏‎‏‎‏‎‎‏‏‎‏‎‏‏‏‎Emergency alert‎‏‎‎‏‎"</string>
+    <string name="emergency_alerts_title" msgid="6605036374197485429">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‏‏‏‎‏‎‏‎‎‏‏‏‎‎‏‏‎‏‏‎‎‏‎‏‏‎‎‏‎‏‏‏‎‎‏‎‎‏‎‎‏‏‎‎‏‏‏‏‏‏‎‏‏‏‎‏‎‏‎Alerts‎‏‎‎‏‎"</string>
+    <string name="notification_channel_broadcast_messages" msgid="880704362482824524">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‎‎‎‎‏‏‏‎‎‎‏‏‏‎‎‎‏‏‏‏‏‏‎‎‏‏‏‎‎‏‎‏‎‏‏‏‏‎‏‏‎‎‏‏‏‏‎‏‎‏‎‏‎‎‏‏‎‎‎Broadcast messages‎‏‎‎‏‎"</string>
+    <string name="notification_channel_emergency_alerts" msgid="5008287980979183617">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‎‏‏‎‎‎‎‎‎‏‎‎‎‎‎‎‏‏‎‏‏‎‎‎‏‎‎‎‏‏‎‏‏‏‎‎‎‏‏‏‏‏‎‏‏‎‎‎‎‎‎‎‎‎‎‎‎‏‎Emergency alerts‎‏‎‎‏‎"</string>
+    <string name="enable_alerts_master_toggle_title" msgid="1457904343636699446">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‎‎‎‏‏‏‎‏‏‏‎‎‎‎‏‎‎‎‏‎‎‎‏‏‎‏‎‏‏‎‏‏‎‏‏‏‏‎‏‏‎‏‏‏‎‏‎‎‏‎‎‏‏‎‏‏‎‎Allow alerts‎‏‎‎‏‎"</string>
+    <string name="enable_alerts_master_toggle_summary" msgid="699552922381916044">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‏‏‎‏‏‎‏‎‏‎‏‎‎‏‏‏‏‏‎‏‎‏‏‎‏‎‎‏‏‎‎‎‎‎‎‏‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‎‎‏‏‎‎‎Receive emergency notifications‎‏‎‎‏‎"</string>
+    <string name="alert_reminder_interval_title" msgid="7466642011937564868">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‏‏‏‎‎‏‏‏‏‎‏‏‎‏‎‏‏‏‎‏‎‏‏‎‎‏‏‎‎‏‎‎‏‎‏‎‎‎‏‎‏‏‏‏‏‎‏‎‎‎‏‏‎‎‎‏‎‎‎Alert reminder sound‎‏‎‎‏‎"</string>
+    <string name="enable_alert_speech_title" msgid="8052104771053526941">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‏‏‏‏‎‏‏‏‏‏‎‏‏‎‏‎‎‏‎‏‎‏‎‎‎‎‎‏‎‎‏‏‏‎‎‏‏‎‏‏‏‏‏‏‎‎‏‏‏‏‏‏‎‎‏‏‏‎‏‎Speak alert message‎‏‎‎‏‎"</string>
+    <string name="enable_alert_speech_summary" msgid="356086178962268831">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‏‏‎‎‏‏‏‏‎‎‎‏‎‎‎‏‎‎‏‎‎‏‏‏‏‏‏‎‎‎‏‏‏‏‏‎‎‏‏‏‏‎‎‎‏‏‏‎‎‎‏‎‏‎‎‏‏‏‏‏‎Use text-to-speech to speak emergency alert messages‎‏‎‎‏‎"</string>
+    <string name="emergency_alert_history_title" msgid="8310173569237268431">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‎‏‏‎‏‎‏‎‎‏‏‏‎‏‎‏‎‏‎‏‏‎‏‎‏‎‏‎‎‎‎‎‎‎‎‎‎‏‏‏‎‏‎‎‎‏‏‎‏‏‏‏‏‎‎‏‏‏‏‎Emergency alert history‎‏‎‎‏‎"</string>
+    <string name="alert_preferences_title" msgid="6001469026393248468">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‎‏‏‎‏‎‎‏‎‎‏‏‎‎‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‎‎‏‎‎‎‏‎‏‎‏‏‎‏‏‏‎‏‎‎‏‎‏‏‎‏‎‏‎‎‎Alert preferences‎‏‎‎‏‎"</string>
+    <string name="enable_etws_test_alerts_title" msgid="3593533226735441539">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‎‎‏‏‏‎‏‏‏‏‎‏‏‎‎‏‎‏‏‎‏‏‎‏‏‏‎‎‎‏‎‏‏‎‎‎‎‏‎‎‎‎‏‎‎‎‏‎‏‏‎‏‎‎‎‎‎‏‏‎ETWS test broadcasts‎‏‎‎‏‎"</string>
+    <string name="enable_etws_test_alerts_summary" msgid="8746155402612927306">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‎‎‏‎‏‏‎‎‎‎‎‏‎‎‏‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‏‎‎‎‏‏‎‏‏‎‏‏‎‏‎‏‏‎‏‎‎‏‎‏‎‎Test broadcasts for Earthquake Tsunami Warning System‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_extreme_threat_alerts_title" msgid="5416260219062637770">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‎‏‏‎‎‏‎‏‎‏‎‎‏‏‎‏‎‏‏‏‏‏‏‏‎‎‏‎‎‎‎‎‎‎‎‎‏‎‎‏‏‏‎‏‏‎‏‎‏‎‎‏‏‎‎‏‎‏‎‎Extreme threats‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_extreme_threat_alerts_summary" msgid="5832146246627518123">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‎‎‎‏‏‏‎‏‏‏‏‏‏‏‏‎‎‏‎‎‎‎‏‏‏‎‎‎‏‏‏‏‏‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‏‏‎‏‎‏‎‏‎‏‏‎Extreme threats to life and property‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_severe_threat_alerts_title" msgid="1066172973703410042">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‏‎‏‏‎‎‏‎‏‏‏‏‎‎‏‏‏‎‏‎‏‎‏‎‏‏‏‎‏‏‎‏‏‎‏‎‏‏‎‏‏‎‏‎‏‏‎‎‎‏‎‏‏‏‏‎‏‎‎Severe threats‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_severe_threat_alerts_summary" msgid="5292443310309039223">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‎‎‏‎‏‏‏‎‎‏‎‏‎‎‎‏‎‎‏‎‎‏‎‎‏‏‎‎‏‏‏‎‎‎‎‏‏‎‏‏‏‎‎‏‏‎‏‏‏‎‎‎‏‏‏‎‏‏‏‎Severe threats to life and property‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_amber_alerts_title" msgid="1475030503498979651">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‎‎‏‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‏‏‎‏‏‎‎‎‏‎‎‏‏‏‏‎‏‏‎‎‎‎‎‏‏‎‎‏‎‏‎‎‎‎‏‏‎AMBER alerts‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_amber_alerts_summary" msgid="4495233280416889667">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‏‏‎‎‏‏‎‎‎‏‎‎‏‎‎‎‏‏‎‏‏‎‏‏‎‏‎‎‎‎‎‏‏‎‏‏‏‎‎‎‎‏‏‎‎‏‏‏‎‏‏‎‏‎‎‎‎‏‏‎Child abduction emergency bulletins‎‏‎‎‏‎"</string>
+    <string name="enable_alert_message_title" msgid="2939830587633599352">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‎‎‏‏‎‎‏‏‎‎‎‏‏‎‎‎‎‎‎‏‎‏‎‏‏‏‏‏‏‏‎‎‏‏‏‎‏‏‏‎‏‎‏‎‎‎‎‎‏‏‎‏‏‏‏‎‎‎‎Alert messages‎‏‎‎‏‎"</string>
+    <string name="enable_alert_message_summary" msgid="6525664541696985610">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‏‎‏‎‎‎‏‏‏‏‏‏‎‏‎‎‎‏‎‏‎‏‎‎‏‏‏‎‎‎‎‎‏‏‎‎‏‏‎‏‏‎‎‎‎‎‏‎‏‎‎‎‎‎‏‎‏‎‎Warn about imminent safety threats‎‏‎‎‏‎"</string>
+    <string name="enable_public_safety_messages_title" msgid="5576770949182656524">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‏‎‏‎‏‏‎‎‏‎‎‏‎‏‎‏‎‏‏‏‎‏‎‎‎‎‏‏‎‎‏‏‎‏‏‏‏‎‎‎‎‎‎‎‎‎‏‎‎‎‎‎‎‎‎‏‏‎‎‎Public safety messages‎‏‎‎‏‎"</string>
+    <string name="enable_public_safety_messages_summary" msgid="7868069748857851521">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‏‎‏‎‎‏‏‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‏‏‏‏‏‏‏‏‏‎‎‎‎‏‎‏‎‏‎‏‏‎‏‎‎‏‏‏‎‏‎‎‎‎‎‎‏‎Recommended actions that can save lives or property‎‏‎‎‏‎"</string>
+    <string name="enable_state_local_test_alerts_title" msgid="1012930918171302720">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‏‎‎‎‎‎‏‏‏‎‏‎‏‎‎‏‏‏‎‏‎‎‏‏‎‏‏‎‏‎‎‏‏‏‏‎‎‏‏‏‎‎‎‏‎‏‏‎‏‏‎‏‎‎‎‎‎‎‎State and local tests‎‏‎‎‏‎"</string>
+    <string name="enable_state_local_test_alerts_summary" msgid="780298327377950187">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‏‎‏‏‎‏‎‏‎‎‎‎‏‎‏‏‎‏‎‎‏‏‎‎‎‏‏‏‎‎‏‏‏‎‎‎‏‏‎‎‎‏‏‎‏‏‎‏‎‏‏‏‏‎‏‎‏‏‎Receive test messages from state and local authorities‎‏‎‎‏‎"</string>
+    <string name="enable_emergency_alerts_message_title" msgid="661894007489847468">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‏‎‎‏‎‏‏‏‏‏‎‎‎‎‏‎‏‎‎‎‏‎‏‏‏‏‏‎‎‏‎‏‏‎‏‏‎‎‏‎‎‎‏‏‎‎‎‎‎‏‎‏‎‏‏‎‎‎Emergency alerts‎‏‎‎‏‎"</string>
+    <string name="enable_emergency_alerts_message_summary" msgid="7574617515441602546">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‎‎‏‎‎‎‏‏‏‏‎‎‏‏‏‎‎‏‎‎‏‏‏‏‏‏‏‏‎‎‎‏‏‎‏‎‏‎‏‏‎‎‏‎‎‏‏‏‏‏‏‏‏‏‏‎‎‏‎‎Warn about life-threatening events‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_test_alerts_title" msgid="1319231576046146463">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‎‏‎‎‏‎‎‏‏‏‎‏‏‎‏‏‎‏‎‎‎‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‏‏‎‏‎‏‏‎‎‏‎‎‏‏‏‎‎‏‏‏‏‏‎Required monthly test‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_test_alerts_summary" msgid="588517403520088484">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‎‎‎‏‎‏‎‏‎‏‏‎‏‎‏‎‏‎‏‏‏‎‏‎‏‏‏‎‏‏‏‎‎‏‏‏‏‎‎‏‏‎‎‏‏‏‎‎‏‏‎‏‎‎‏‎‎‎Receive test messages for the safety alert system‎‏‎‎‏‎"</string>
+    <string name="enable_alert_vibrate_title" msgid="982817538757982090">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‎‏‏‎‏‎‎‎‏‏‏‎‏‎‏‎‏‏‎‏‎‏‏‎‎‎‎‎‏‎‎‏‎‎‏‎‏‏‎‎‎‏‎‏‏‏‎‏‏‏‏‎‎‎‏‎‏‎‎Vibrate‎‏‎‎‏‎"</string>
+    <string name="use_full_volume_title" msgid="8581439612945182255">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‏‏‎‎‎‏‎‏‏‏‎‏‏‎‎‏‎‏‏‏‎‏‏‏‎‎‏‎‎‎‏‏‏‏‎‎‎‏‎‎‎‎‎‎‏‎‏‏‏‎‎‎‏‎‏‏‏‏‎Use full volume‎‏‎‎‏‎"</string>
+    <string name="use_full_volume_summary" msgid="3560764277281704697">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‎‎‏‎‏‏‎‏‎‏‎‎‏‏‎‎‎‎‎‎‎‏‏‏‏‏‎‏‎‎‏‎‏‎‎‎‏‎‎‎‏‏‎‏‏‎‎‏‎‏‎‏‏‏‏‏‎‎‏‎Ignore other volume and Do Not Disturb preferences‎‏‎‎‏‎"</string>
+    <string name="enable_area_update_info_alerts_title" msgid="3442042268424617226">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‏‏‏‏‏‎‎‎‏‎‎‏‎‎‏‎‏‏‏‎‎‏‏‎‏‎‎‏‏‎‏‎‏‏‏‎‏‎‏‎‏‎‎‎‎‏‏‏‏‎‏‎‎‎‎‏‎‏‎‎Area update broadcasts‎‏‎‎‏‎"</string>
+    <string name="enable_area_update_info_alerts_summary" msgid="6437816607144264910">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‎‏‎‏‎‏‎‏‏‏‏‎‏‏‏‎‎‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‎‏‏‏‏‏‎‎‎‏‏‏‎‎‎‏‎‎‏‏‎‎‏‏‏‎‎Show update information in SIM status‎‏‎‎‏‎"</string>
+    <string name="category_dev_settings_title" msgid="6194393458398329994">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‏‎‏‏‏‏‏‎‏‏‎‏‏‏‎‏‎‎‎‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‏‏‎‏‎‏‎‏‏‏‏‏‏‏‎‎‏‎‎‎‏‎‏‎‎Developer options‎‏‎‎‏‎"</string>
+    <string name="cmas_category_heading" msgid="3923503130776640717">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‏‏‎‎‏‏‏‎‎‏‏‎‎‎‏‎‏‎‏‎‏‎‎‏‏‏‏‎‏‎‏‎‎‎‏‎‎‎‏‎‎‏‏‎‎‎‏‏‏‎‎‏‏‎‎‏‏‎‏‎Alert Category:‎‏‎‎‏‎"</string>
+    <string name="cmas_category_geo" msgid="4979494217069688527">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‎‏‎‎‎‏‏‎‏‎‏‎‏‏‎‏‏‏‏‎‎‏‏‎‏‏‏‎‏‎‎‎‎‏‏‎‎‎‏‏‎‎‏‏‏‏‎‏‏‎‏‏‎‎‏‏‏‏‎Geophysical‎‏‎‎‏‎"</string>
+    <string name="cmas_category_met" msgid="7563732573851773537">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‎‎‎‏‏‏‏‎‏‏‏‏‏‎‎‎‏‏‎‏‎‏‏‎‎‏‏‏‎‎‏‏‎‎‎‎‏‏‏‏‏‎‎‎‏‏‎‏‎‏‎‎‏‏‎‎‎‎‏‎Meteorological‎‏‎‎‏‎"</string>
+    <string name="cmas_category_safety" msgid="2986472639641883453">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‎‏‎‏‏‏‎‎‏‎‎‎‎‏‎‏‎‏‎‎‎‎‏‎‏‎‏‏‏‏‎‎‎‏‎‏‎‎‎‏‏‏‎‎‎‏‎‎‏‏‎‎‏‏‏‏‎‏‎Safety‎‏‎‎‏‎"</string>
+    <string name="cmas_category_security" msgid="2549520159044403704">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‎‎‏‏‎‏‏‎‎‎‎‏‏‎‏‏‎‏‏‏‎‎‎‏‎‎‏‏‏‎‏‎‎‎‏‏‏‎‎‎‏‎‏‎‎‎‎‎‎‎‎‏‏‏‏‏‏‎‎‎‎Security‎‏‎‎‏‎"</string>
+    <string name="cmas_category_rescue" msgid="4907571719983321086">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‎‎‎‎‎‏‏‎‏‏‎‎‏‏‎‎‏‎‎‏‏‏‏‎‏‎‎‏‎‏‎‏‏‏‎‏‎‏‏‎‏‏‏‏‎‎‏‏‏‏‏‏‏‏‏‏‏‎‎Rescue‎‏‎‎‏‎"</string>
+    <string name="cmas_category_fire" msgid="3331981591918341119">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‏‏‎‎‎‏‏‏‏‎‏‏‎‎‏‎‎‏‏‏‎‎‏‏‎‏‎‏‎‏‏‎‎‎‎‎‏‏‎‏‏‏‎‏‏‏‎‏‎‏‏‏‏‏‏‏‏‏‏‎Fire‎‏‎‎‏‎"</string>
+    <string name="cmas_category_health" msgid="312569774587117324">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‏‏‎‎‎‏‎‏‎‏‏‎‎‏‏‏‏‎‎‎‏‎‎‎‏‏‎‏‏‏‎‎‏‏‎‏‏‏‎‎‏‎‎‏‏‏‏‏‏‏‏‏‎‎‎‎‏‏‎‎‎Health‎‏‎‎‏‎"</string>
+    <string name="cmas_category_env" msgid="9213088574227522961">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‏‏‏‏‎‏‏‎‏‏‎‏‏‏‎‏‏‏‎‎‏‏‏‏‏‎‏‏‎‎‎‏‏‏‎‎‎‎‏‎‏‎‏‎‏‎‎‏‎‏‏‎‎‏‎‎‎‏‎Environmental‎‏‎‎‏‎"</string>
+    <string name="cmas_category_transport" msgid="5014901635987361642">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‎‏‏‎‎‏‏‎‎‎‏‎‎‎‎‎‏‎‎‏‏‏‎‏‏‏‏‏‎‎‎‏‎‎‏‎‎‎‏‏‎‏‎‏‎‎‏‏‏‏‎‏‏‎‏‎‏‎‎Transportation‎‏‎‎‏‎"</string>
+    <string name="cmas_category_infra" msgid="3558151044446851398">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‎‎‏‎‏‏‎‎‎‎‏‎‎‎‏‎‏‏‏‏‎‎‎‎‏‎‏‏‏‏‎‎‏‎‏‎‏‏‏‎‎‎‎‏‎‎‏‎‎‎‏‎‏‎‎‎‏‏‎‎Infrastructure‎‏‎‎‏‎"</string>
+    <string name="cmas_category_cbrne" msgid="240421557913603971">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‎‏‏‎‏‎‏‎‏‏‎‎‎‏‎‎‏‏‎‎‎‏‎‎‎‏‎‎‎‎‏‎‎‏‏‏‏‎‎‏‎‎‏‎‏‎‎‏‏‏‏‏‎‎‎‎‎‏‏‎Chemical/Biological/Nuclear/Explosive‎‏‎‎‏‎"</string>
+    <string name="cmas_category_other" msgid="6158932360790744360">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‏‎‏‎‏‏‏‏‎‎‎‏‏‏‎‏‏‎‎‎‏‎‏‏‎‎‏‏‏‎‎‏‎‎‎‏‎‎‏‎‏‏‏‎‎‎‏‏‏‎‏‎‎‏‎‏‎‎‎‎Other‎‏‎‎‏‎"</string>
+    <string name="cmas_response_heading" msgid="4205379547245540163">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‏‎‎‏‎‏‏‏‎‎‏‎‎‎‎‎‏‎‎‏‏‎‏‎‏‎‏‏‎‎‏‎‏‏‎‏‎‎‎‎‏‏‎‏‏‏‎‎‏‏‎‏‎‎‎‎‏‏‎Response Type:‎‏‎‎‏‎"</string>
+    <string name="cmas_response_shelter" msgid="7301175579079615909">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‎‏‎‏‎‏‎‎‏‎‏‏‏‏‏‏‎‎‏‎‎‎‎‎‎‏‏‎‎‎‎‎‎‏‏‏‎‎‏‏‎‎‎‎‏‎‎‏‎‏‏‎‏‎‎‏‎‏‎Shelter‎‏‎‎‏‎"</string>
+    <string name="cmas_response_evacuate" msgid="5833170084430021095">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‎‎‎‏‏‏‏‎‎‏‏‏‎‎‏‎‏‎‏‎‏‎‎‏‎‎‏‎‏‎‎‏‏‎‎‏‏‏‎‎‎‏‏‎‎‎‎‎‎‎‏‏‏‏‎‎‏‏‏‎Evacuate‎‏‎‎‏‎"</string>
+    <string name="cmas_response_prepare" msgid="8428073909753758319">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‎‎‏‏‏‏‎‏‏‎‏‎‎‎‏‎‎‎‏‎‎‏‎‎‎‏‎‎‏‎‏‎‏‏‏‎‎‏‏‏‎‎‎‏‎‎‎‎‏‎‎‏‏‎‏‏‏‏‎Prepare‎‏‎‎‏‎"</string>
+    <string name="cmas_response_execute" msgid="284719420769568493">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‎‏‏‏‏‏‏‎‎‏‏‏‎‎‎‎‏‏‎‏‏‎‎‏‏‎‏‏‏‏‏‏‏‏‏‏‎‏‎‎‎‎‎‎‎‎‎‎‏‏‎‏‏‏‎‏‏‎‏‎Execute‎‏‎‎‏‎"</string>
+    <string name="cmas_response_monitor" msgid="681400164440495749">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‎‎‏‎‏‏‏‎‏‎‎‏‏‎‏‎‎‎‏‏‏‎‏‎‏‏‎‏‏‎‏‏‏‏‏‎‏‎‏‏‏‏‎‎‏‎‏‎‏‏‎‏‎‎‎‎‏‎‏‎Monitor‎‏‎‎‏‎"</string>
+    <string name="cmas_response_avoid" msgid="156419597612629270">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‎‏‎‎‎‏‎‏‎‏‏‏‎‏‏‎‏‏‎‏‏‎‎‏‎‏‏‏‎‎‏‎‏‏‏‏‎‎‏‏‏‎‎‎‎‏‏‏‏‎‏‎‎‎‏‎‏‏‎‎Avoid‎‏‎‎‏‎"</string>
+    <string name="cmas_response_assess" msgid="9043534222710563415">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‎‏‏‎‎‎‎‎‎‏‎‎‎‏‎‏‏‎‎‏‏‏‎‏‎‏‏‏‏‏‏‎‏‎‏‏‎‎‎‎‎‎‎‏‏‎‎‎‏‎‎‏‎‏‎‏‏‏‎Assess‎‏‎‎‏‎"</string>
+    <string name="cmas_response_none" msgid="5149009359674452959">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‏‏‎‏‏‏‎‏‎‎‏‏‏‏‎‏‎‎‏‎‏‏‏‎‏‏‎‏‏‎‎‏‎‎‏‎‎‎‏‏‏‎‏‎‏‎‏‎‏‏‏‏‎‏‏‏‏‏‎None‎‏‎‎‏‎"</string>
+    <string name="cmas_severity_heading" msgid="8437057117822305243">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‎‏‎‎‎‏‎‏‏‎‎‏‏‏‎‎‏‎‏‎‏‏‏‏‏‏‎‏‎‎‏‎‎‏‎‏‏‏‏‎‏‎‏‏‎‎‎‎‏‏‏‏‎‏‏‎‏‏‎Severity:‎‏‎‎‏‎"</string>
+    <string name="cmas_severity_extreme" msgid="1312013282860183082">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‎‏‎‎‎‏‏‎‏‎‏‎‎‏‏‎‏‎‏‎‎‏‎‎‎‎‏‎‎‏‎‎‎‎‏‎‏‏‏‏‎‎‏‏‎‏‎‏‏‏‎‎‎‏‎‏‎‏‎‎Extreme‎‏‎‎‏‎"</string>
+    <string name="cmas_severity_severe" msgid="7504359209737074524">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‎‎‎‎‎‏‎‎‏‎‎‏‏‎‏‎‏‏‎‏‏‏‏‎‎‎‏‎‎‎‎‎‎‏‏‎‏‎‎‏‏‏‏‎‎‏‏‎‏‏‏‎‏‎‏‏‏‎‎‎Severe‎‏‎‎‏‎"</string>
+    <string name="cmas_urgency_heading" msgid="8218282767913431492">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‎‏‎‎‎‎‎‏‏‎‏‎‎‏‏‎‏‎‎‏‎‏‎‎‎‏‎‏‎‎‏‎‏‎‏‎‎‏‏‎‎‎‏‏‎‏‏‎‎‎‏‏‏‎‎‎‏‎‎‎Urgency:‎‏‎‎‏‎"</string>
+    <string name="cmas_urgency_immediate" msgid="1577485208196449288">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‏‏‏‏‎‎‏‎‎‎‏‎‏‏‎‏‎‎‏‏‏‎‎‎‎‎‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‎‎‎‏‎‎‎‎‎‎‏‎‎‎‎Immediate‎‏‎‎‏‎"</string>
+    <string name="cmas_urgency_expected" msgid="6830831119872375936">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‏‏‎‏‏‎‎‏‎‏‏‏‏‏‏‏‏‎‎‏‎‏‏‎‏‏‎‎‏‎‏‎‏‎‎‏‎‏‎‏‏‎‏‎‎‏‏‏‏‎‎‏‎‎‎‎‎‎‎‎Expected‎‏‎‎‏‎"</string>
+    <string name="cmas_certainty_heading" msgid="8374669249736439193">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‎‏‎‎‎‎‏‏‏‎‎‎‏‏‎‎‏‏‎‏‎‏‎‎‏‏‏‏‏‏‎‎‏‏‎‏‏‏‏‎‏‏‎‏‎‏‏‏‎‏‎‏‏‎‎‏‏‎‎‏‎Certainty:‎‏‎‎‏‎"</string>
+    <string name="cmas_certainty_observed" msgid="3668549749352106472">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‎‏‎‏‏‏‎‏‎‎‏‎‏‎‎‏‏‏‎‏‎‎‎‏‏‏‏‎‏‏‎‏‎‏‎‏‏‏‏‏‎‏‏‎‎‎‎‏‎‎‏‏‏‏‎‏‎‎‎‎Observed‎‏‎‎‏‎"</string>
+    <string name="cmas_certainty_likely" msgid="4254497828943291749">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‏‏‎‎‎‎‏‎‏‏‎‎‎‎‎‎‏‏‎‎‏‏‏‎‏‏‏‏‏‎‏‏‏‏‏‏‎‏‎‏‎‏‏‏‏‎‎‏‎‏‎‏‏‎‎‏‎‏‎Likely‎‏‎‎‏‎"</string>
+    <string name="delivery_time_heading" msgid="5980836543433619329">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‎‎‏‏‎‎‎‎‎‎‎‎‎‎‏‏‎‎‏‏‎‎‏‎‎‏‎‏‏‏‏‏‏‏‏‏‎‎‎‎‏‎‎‏‎‎‏‏‎‏‏‏‏‎‎‎‎‎‎‏‎Received:‎‏‎‎‏‎"</string>
+    <string name="notification_multiple" msgid="5121978148152124860">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‏‏‏‎‎‎‏‎‏‎‎‏‏‏‎‏‎‏‏‏‏‏‏‏‏‎‏‎‎‎‎‎‎‏‏‏‏‏‎‏‎‏‏‏‎‏‎‎‏‎‏‏‎‏‏‏‏‎‎‎‎‏‎‎‏‏‎<xliff:g id="COUNT">%s</xliff:g>‎‏‎‎‏‏‏‎ unread alerts.‎‏‎‎‏‎"</string>
+    <string name="notification_multiple_title" msgid="1523638925739947855">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‏‎‎‏‎‎‏‎‏‎‎‎‎‏‏‎‏‏‎‎‎‎‏‏‏‏‏‎‎‎‏‏‎‏‎‏‎‏‏‏‏‏‏‏‏‎‏‏‏‎‏‎‎‏‏‏‏‎New alerts‎‏‎‎‏‎"</string>
+    <string name="show_cmas_opt_out_summary" msgid="6926059266585295440">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‎‎‎‎‎‎‏‏‏‏‎‎‏‎‎‏‏‏‎‎‎‏‏‎‏‎‏‎‎‎‎‎‏‏‎‎‎‎‏‏‏‏‎‎‏‏‏‏‎‏‎‎‏‎‏‎‎‎‎‎Show an opt-out dialog after displaying the first alert (other than Presidential Alert).‎‏‎‎‏‎"</string>
+    <string name="show_cmas_opt_out_title" msgid="9182104842820171132">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‏‏‎‏‏‎‏‏‎‏‎‏‏‎‎‎‏‏‏‎‏‏‎‏‎‎‏‎‏‎‎‏‏‎‏‎‏‏‎‎‎‎‏‎‏‏‎‏‎‏‎‏‏‏‏‏‎‎‎Show opt-out dialog‎‏‎‎‏‎"</string>
+    <string name="cmas_opt_out_dialog_text" msgid="7529010670998259128">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‏‎‎‎‎‏‏‏‏‏‎‎‎‏‏‎‏‎‏‏‎‏‎‏‎‎‎‎‏‎‎‏‏‎‏‎‏‏‏‏‏‎‏‎‎‎‎‎‎‏‎‏‏‎‏‏‏‎‎‎‎You are currently receiving Emergency Alerts. Would you like to continue receiving Emergency Alerts?‎‏‎‎‏‎"</string>
+    <string name="cmas_opt_out_button_yes" msgid="7248930667195432936">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‎‎‏‎‎‏‏‎‎‏‎‏‏‎‎‎‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‎‏‏‏‎‎‏‏‏‏‎‎‏‏‏‏‏‏‏‏‏‏‎‏‎‎‎‎Yes‎‏‎‎‏‎"</string>
+    <string name="cmas_opt_out_button_no" msgid="3110484064328538553">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‎‏‎‏‏‎‎‏‎‏‎‏‎‏‎‏‎‏‎‎‎‏‏‎‎‎‏‏‎‏‏‎‎‎‏‎‏‏‏‎‏‎‎‎‎‏‏‎‏‏‎‎‏‏‎‏‏‏‎‎‏‎No‎‏‎‎‏‎"</string>
+    <string name="cb_list_activity_title" msgid="1433502151877791724">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‏‎‎‏‏‏‏‏‎‎‏‎‎‏‏‎‏‎‎‏‎‏‎‎‏‏‏‎‎‏‎‏‏‎‎‏‏‎‏‏‎‏‏‏‏‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‎‎Emergency alert history‎‏‎‎‏‎"</string>
   <string-array name="alert_reminder_interval_entries">
-    <item msgid="6595211083588795160">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‏‏‏‎‎‎‎‏‏‎‏‏‏‎‎‏‎‏‏‎‎‎‏‎‎‏‏‎‏‎‎‎‎‎‎‎‎‎‎‏‏‏‎‏‏‏‏‎‏‏‎‎‎‏‏‎‎‎‎Once‎‏‎‎‏‎"</item>
-    <item msgid="9097229303902157183">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‏‏‏‏‏‏‏‎‏‏‎‎‏‏‏‎‏‏‎‏‎‎‏‎‎‏‎‏‏‎‏‎‏‏‏‎‎‏‎‏‏‏‎‎‏‎‏‏‏‏‏‏‏‎Every 2 minutes‎‏‎‎‏‎"</item>
-    <item msgid="5718214950343391480">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‏‏‏‎‏‎‏‏‎‏‏‎‎‏‎‏‏‏‎‎‎‏‏‎‎‏‏‎‏‏‏‎‎‎‎‏‎‎‎‏‎‏‏‏‎‏‏‏‎‎‎‏‏‏‏‏‎‎‎‎Every 5 minutes‎‏‎‎‏‎"</item>
-    <item msgid="3863339891188103437">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‏‎‏‏‎‎‏‏‏‎‏‎‏‎‏‎‏‏‏‎‎‏‎‏‎‎‏‏‎‎‎‎‏‎‎‏‏‎‏‏‏‎‎‏‏‏‏‏‎‎‏‎‎‎‎‏‏‎‏‎Every 15 minutes‎‏‎‎‏‎"</item>
-    <item msgid="6868848414437854609">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‏‏‏‎‏‎‏‎‎‏‏‎‎‎‎‏‏‎‏‎‎‏‏‏‏‎‏‎‏‏‏‏‏‏‎‏‎‎‏‎‎‏‎‏‏‎‎‏‏‎‏‏‎‎‏‎‎‎‏‎Off‎‏‎‎‏‎"</item>
+    <item msgid="6595211083588795160">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‏‏‏‎‎‎‎‏‏‎‏‏‏‎‎‏‎‏‏‎‎‎‏‎‎‏‏‎‏‎‎‎‎‎‎‎‎‎‎‏‏‏‎‏‏‏‏‎‏‏‎‎‎‏‏‎‎‎‎Once‎‏‎‎‏‎"</item>
+    <item msgid="9097229303902157183">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‏‏‏‏‏‏‏‎‏‏‎‎‏‏‏‎‏‏‎‏‎‎‏‎‎‏‎‏‏‎‏‎‏‏‏‎‎‏‎‏‏‏‎‎‏‎‏‏‏‏‏‏‏‎Every 2 minutes‎‏‎‎‏‎"</item>
+    <item msgid="5718214950343391480">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‏‏‏‏‎‏‎‏‏‎‏‏‎‎‏‎‏‏‏‎‎‎‏‏‎‎‏‏‎‏‏‏‎‎‎‎‏‎‎‎‏‎‏‏‏‎‏‏‏‎‎‎‏‏‏‏‏‎‎‎‎Every 5 minutes‎‏‎‎‏‎"</item>
+    <item msgid="3863339891188103437">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‎‏‎‏‏‎‎‏‏‏‎‏‎‏‎‏‎‏‏‏‎‎‏‎‏‎‎‏‏‎‎‎‎‏‎‎‏‏‎‏‏‏‎‎‏‏‏‏‏‎‎‏‎‎‎‎‏‏‎‏‎Every 15 minutes‎‏‎‎‏‎"</item>
+    <item msgid="6868848414437854609">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‏‏‏‎‏‎‏‎‎‏‏‎‎‎‎‏‏‎‏‎‎‏‏‏‏‎‏‎‏‏‏‏‏‏‎‏‎‎‏‎‎‏‎‏‏‎‎‏‏‎‏‏‎‎‏‎‎‎‏‎Off‎‏‎‎‏‎"</item>
   </string-array>
-    <string name="emergency_alert_settings_title_watches" msgid="4477073412799894883">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‏‏‎‎‎‏‎‎‎‎‏‏‏‎‎‎‎‏‎‏‎‎‎‏‏‎‎‎‏‏‎‎‎‏‏‎‏‎‎‏‎‎‎‏‎‎‏‎‏‎‏‎‏‏‎‎‎‏‏‎Wireless emergency alerts‎‏‎‎‏‎"</string>
+    <string name="emergency_alert_settings_title_watches" msgid="4477073412799894883">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‏‏‎‎‎‏‎‎‎‎‏‏‏‎‎‎‎‏‎‏‎‎‎‏‏‎‎‎‏‏‎‎‎‏‏‎‏‎‎‏‎‎‎‏‎‎‏‎‏‎‏‎‏‏‎‎‎‏‏‎Wireless emergency alerts‎‏‎‎‏‎"</string>
 </resources>
diff --git a/res/values-gl/strings.xml b/res/values-gl/strings.xml
index 5cef557f..a1b35eff 100644
--- a/res/values-gl/strings.xml
+++ b/res/values-gl/strings.xml
@@ -17,7 +17,7 @@
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <string name="app_label" msgid="7559008249836700599">"Difusións móbiles"</string>
-    <string name="sms_cb_settings" msgid="3795852619005211509">"Alertas de urxencia"</string>
+    <string name="sms_cb_settings" msgid="3795852619005211509">"Alertas de emerxencia"</string>
     <string name="cell_broadcast_settings_not_available" msgid="2612364350745967915">"A configuración da difusión móbil non está dispoñible para este usuario"</string>
     <string name="button_dismiss" msgid="1234221657930516287">"Aceptar"</string>
     <string name="no_cell_broadcasts" msgid="5409324337492758562">"Non hai alertas activas na túa zona. Podes cambiar a configuración das alertas a través da opción do menú Configuración."</string>
@@ -34,29 +34,29 @@
     <string name="etws_tsunami_warning" msgid="6173964105145900312">"Aviso de tsunami"</string>
     <string name="etws_earthquake_and_tsunami_warning" msgid="662449983177407681">"Aviso de terremoto e tsunami"</string>
     <string name="etws_test_message" msgid="8447820262584381894">"Mensaxe de proba do ETWS"</string>
-    <string name="etws_other_emergency_type" msgid="5233080551309721499">"Aviso de urxencia"</string>
+    <string name="etws_other_emergency_type" msgid="5233080551309721499">"Aviso de emerxencia"</string>
     <string name="cmas_presidential_level_alert" msgid="1209234030582361001">"Alerta presidencial"</string>
-    <string name="cmas_extreme_alert" msgid="2588720613319969289">"Alerta de urxencia: extrema"</string>
-    <string name="cmas_extreme_immediate_observed_alert" msgid="2328845915287460780">"Alerta de urxencia: extrema"</string>
-    <string name="cmas_extreme_immediate_likely_alert" msgid="1859702950323471778">"Alerta de urxencia: extrema"</string>
-    <string name="cmas_severe_alert" msgid="4135809475315826913">"Alerta de urxencia: grave"</string>
+    <string name="cmas_extreme_alert" msgid="2588720613319969289">"Alerta de emerxencia: extrema"</string>
+    <string name="cmas_extreme_immediate_observed_alert" msgid="2328845915287460780">"Alerta de emerxencia: extrema"</string>
+    <string name="cmas_extreme_immediate_likely_alert" msgid="1859702950323471778">"Alerta de emerxencia: extrema"</string>
+    <string name="cmas_severe_alert" msgid="4135809475315826913">"Alerta de emerxencia: grave"</string>
     <string name="cmas_amber_alert" msgid="6154867710264778887">"Secuestro de menores (alerta AMBER)"</string>
     <string name="cmas_required_monthly_test" msgid="1890205712251132193">"Proba mensual obrigatoria"</string>
-    <string name="cmas_exercise_alert" msgid="2892255514938370321">"Alerta de urxencia (simulacro)"</string>
-    <string name="cmas_operator_defined_alert" msgid="8755372450810011476">"Alerta de urxencia (operador)"</string>
+    <string name="cmas_exercise_alert" msgid="2892255514938370321">"Alerta de emerxencia (simulacro)"</string>
+    <string name="cmas_operator_defined_alert" msgid="8755372450810011476">"Alerta de emerxencia (operador)"</string>
     <string name="cb_other_message_identifiers" msgid="5790068194529377210">"Mensaxes de emisión"</string>
     <string name="public_safety_message" msgid="9119928798786998252">"Mensaxe de seguranza pública"</string>
     <string name="state_local_test_alert" msgid="8003145745857480200">"Probas procedentes de autoridades locais/estatais"</string>
-    <string name="emergency_alert" msgid="624783871477634263">"Alerta de urxencia"</string>
+    <string name="emergency_alert" msgid="624783871477634263">"Alerta de emerxencia"</string>
     <string name="emergency_alerts_title" msgid="6605036374197485429">"Alertas"</string>
     <string name="notification_channel_broadcast_messages" msgid="880704362482824524">"Mensaxes de emisión"</string>
-    <string name="notification_channel_emergency_alerts" msgid="5008287980979183617">"Alertas de urxencia"</string>
+    <string name="notification_channel_emergency_alerts" msgid="5008287980979183617">"Alertas de emerxencia"</string>
     <string name="enable_alerts_master_toggle_title" msgid="1457904343636699446">"Permitir alertas"</string>
-    <string name="enable_alerts_master_toggle_summary" msgid="699552922381916044">"Recibir notificacións de urxencia"</string>
+    <string name="enable_alerts_master_toggle_summary" msgid="699552922381916044">"Recibir notificacións de emerxencia"</string>
     <string name="alert_reminder_interval_title" msgid="7466642011937564868">"Son do recordatorio de alerta"</string>
     <string name="enable_alert_speech_title" msgid="8052104771053526941">"Ler mensaxe de alerta"</string>
-    <string name="enable_alert_speech_summary" msgid="356086178962268831">"Usa a síntese de voz para ler as mensaxes de alertas de urxencia"</string>
-    <string name="emergency_alert_history_title" msgid="8310173569237268431">"Historial de alertas de urxencia"</string>
+    <string name="enable_alert_speech_summary" msgid="356086178962268831">"Usa a síntese de voz para ler as mensaxes de alertas de emerxencia"</string>
+    <string name="emergency_alert_history_title" msgid="8310173569237268431">"Historial de alertas de emerxencia"</string>
     <string name="alert_preferences_title" msgid="6001469026393248468">"Preferencias de alertas"</string>
     <string name="enable_etws_test_alerts_title" msgid="3593533226735441539">"Emisións de proba do ETWS"</string>
     <string name="enable_etws_test_alerts_summary" msgid="8746155402612927306">"Emisións de proba do sistema de avisos de terremotos e tsunamis (ETWS)"</string>
@@ -65,14 +65,14 @@
     <string name="enable_cmas_severe_threat_alerts_title" msgid="1066172973703410042">"Ameazas graves"</string>
     <string name="enable_cmas_severe_threat_alerts_summary" msgid="5292443310309039223">"Ameazas graves para vida e a propiedade"</string>
     <string name="enable_cmas_amber_alerts_title" msgid="1475030503498979651">"Alertas AMBER"</string>
-    <string name="enable_cmas_amber_alerts_summary" msgid="4495233280416889667">"Avisos de urxencia de secuestro de menores"</string>
+    <string name="enable_cmas_amber_alerts_summary" msgid="4495233280416889667">"Avisos de emerxencia de secuestro de menores"</string>
     <string name="enable_alert_message_title" msgid="2939830587633599352">"Mensaxes de alerta"</string>
     <string name="enable_alert_message_summary" msgid="6525664541696985610">"Recibir advertencias de ameazas de seguranza inminentes"</string>
     <string name="enable_public_safety_messages_title" msgid="5576770949182656524">"Mensaxes de seguranza pública"</string>
     <string name="enable_public_safety_messages_summary" msgid="7868069748857851521">"Accións recomendadas que poden salvar vidas ou propiedades"</string>
     <string name="enable_state_local_test_alerts_title" msgid="1012930918171302720">"Probas procedentes de autoridades locais e estatais"</string>
     <string name="enable_state_local_test_alerts_summary" msgid="780298327377950187">"Recibir mensaxes de proba procedentes de autoridades locais e estatais"</string>
-    <string name="enable_emergency_alerts_message_title" msgid="661894007489847468">"Alertas de urxencia"</string>
+    <string name="enable_emergency_alerts_message_title" msgid="661894007489847468">"Alertas de emerxencia"</string>
     <string name="enable_emergency_alerts_message_summary" msgid="7574617515441602546">"Recibir advertencias de eventos que poñen a vida en perigo"</string>
     <string name="enable_cmas_test_alerts_title" msgid="1319231576046146463">"Proba mensual obrigatoria"</string>
     <string name="enable_cmas_test_alerts_summary" msgid="588517403520088484">"Recibir mensaxes de proba para o sistema de alertas de seguranza"</string>
@@ -118,10 +118,10 @@
     <string name="notification_multiple_title" msgid="1523638925739947855">"Alertas novas"</string>
     <string name="show_cmas_opt_out_summary" msgid="6926059266585295440">"Mostra un cadro de diálogo de desactivación despois da primeira alerta (agás alerta presidencial)."</string>
     <string name="show_cmas_opt_out_title" msgid="9182104842820171132">"Mostrar un diálogo de desactivación"</string>
-    <string name="cmas_opt_out_dialog_text" msgid="7529010670998259128">"Actualmente recibes alertas de urxencia. Queres seguir recibindo alertas de urxencia?"</string>
+    <string name="cmas_opt_out_dialog_text" msgid="7529010670998259128">"Actualmente recibes alertas de emerxencia. Queres seguir recibindo alertas de emerxencia?"</string>
     <string name="cmas_opt_out_button_yes" msgid="7248930667195432936">"Si"</string>
     <string name="cmas_opt_out_button_no" msgid="3110484064328538553">"Non"</string>
-    <string name="cb_list_activity_title" msgid="1433502151877791724">"Historial de alertas de urxencia"</string>
+    <string name="cb_list_activity_title" msgid="1433502151877791724">"Historial de alertas de emerxencia"</string>
   <string-array name="alert_reminder_interval_entries">
     <item msgid="6595211083588795160">"Unha vez"</item>
     <item msgid="9097229303902157183">"Cada 2 minutos"</item>
@@ -129,5 +129,5 @@
     <item msgid="3863339891188103437">"Cada 15 minutos"</item>
     <item msgid="6868848414437854609">"Desactivado"</item>
   </string-array>
-    <string name="emergency_alert_settings_title_watches" msgid="4477073412799894883">"Alertas de urxencia sen fíos"</string>
+    <string name="emergency_alert_settings_title_watches" msgid="4477073412799894883">"Alertas de emerxencia sen fíos"</string>
 </resources>
diff --git a/res/values-ko/strings.xml b/res/values-ko/strings.xml
index 1cc7cafe..d53c81e8 100644
--- a/res/values-ko/strings.xml
+++ b/res/values-ko/strings.xml
@@ -78,7 +78,7 @@
     <string name="enable_cmas_test_alerts_summary" msgid="588517403520088484">"안전 경보 시스템의 테스트 메시지 수신"</string>
     <string name="enable_alert_vibrate_title" msgid="982817538757982090">"진동"</string>
     <string name="use_full_volume_title" msgid="8581439612945182255">"최대 볼륨 사용"</string>
-    <string name="use_full_volume_summary" msgid="3560764277281704697">"기타 볼륨 및 알림 일시중지 환경설정 무시"</string>
+    <string name="use_full_volume_summary" msgid="3560764277281704697">"기타 볼륨 및 방해 금지 모드 환경설정 무시"</string>
     <string name="enable_area_update_info_alerts_title" msgid="3442042268424617226">"지역 업데이트 방송"</string>
     <string name="enable_area_update_info_alerts_summary" msgid="6437816607144264910">"SIM 상태에 업데이트 정보 표시"</string>
     <string name="category_dev_settings_title" msgid="6194393458398329994">"개발자 옵션"</string>
diff --git a/res/values-mcc246-en-rXC/strings.xml b/res/values-mcc246-en-rXC/strings.xml
index 952e9951..449eabfa 100644
--- a/res/values-mcc246-en-rXC/strings.xml
+++ b/res/values-mcc246-en-rXC/strings.xml
@@ -16,7 +16,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="enable_cmas_extreme_threat_alerts_title" msgid="7485388692049524195">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‏‏‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‏‎‏‎‏‏‎‏‏‎‎‏‎‏‎‏‏‎‏‎‏‎‏‏‎‎‏‏‏‎‏‎‏‏‏‏‎‎‎‏‏‎LT-Alert Level 2‎‏‎‎‏‎"</string>
-    <string name="enable_cmas_severe_threat_alerts_title" msgid="1140833576401868336">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‏‏‏‏‎‏‎‏‎‏‎‎‎‎‏‏‏‎‎‎‎‏‎‏‏‏‏‏‏‎‎‏‎‏‎‏‎‎‎‏‎‏‏‎‎‏‎‏‏‎‎‎‏‏‎‎‎‎‎LT-Alert Level 3‎‏‎‎‏‎"</string>
-    <string name="enable_cmas_amber_alerts_title" msgid="4890124669673498133">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‎‏‏‏‏‎‏‏‏‎‏‎‎‏‏‎‏‏‎‎‏‏‏‏‎‏‎‎‏‏‎‎‎‏‏‎‏‎‎‎‎‏‏‎‏‏‎‎‏‏‎‎‎‎‏‎‏‎‏‎LT-Amber‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_extreme_threat_alerts_title" msgid="7485388692049524195">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‏‏‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‏‎‏‎‏‏‎‏‏‎‎‏‎‏‎‏‏‎‏‎‏‎‏‏‎‎‏‏‏‎‏‎‏‏‏‏‎‎‎‏‏‎LT-Alert Level 2‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_severe_threat_alerts_title" msgid="1140833576401868336">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‎‏‏‏‏‏‏‎‏‎‏‎‏‎‎‎‎‏‏‏‎‎‎‎‏‎‏‏‏‏‏‏‎‎‏‎‏‎‏‎‎‎‏‎‏‏‎‎‏‎‏‏‎‎‎‏‏‎‎‎‎‎LT-Alert Level 3‎‏‎‎‏‎"</string>
+    <string name="enable_cmas_amber_alerts_title" msgid="4890124669673498133">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‎‎‎‏‏‏‏‎‏‏‏‎‏‎‎‏‏‎‏‏‎‎‏‏‏‏‎‏‎‎‏‏‎‎‎‏‏‎‏‎‎‎‎‏‏‎‏‏‎‎‏‏‎‎‎‎‏‎‏‎‏‎LT-Amber‎‏‎‎‏‎"</string>
 </resources>
diff --git a/res/values-mcc450-en-rXC/strings.xml b/res/values-mcc450-en-rXC/strings.xml
index a4afd980..8cc1db7f 100644
--- a/res/values-mcc450-en-rXC/strings.xml
+++ b/res/values-mcc450-en-rXC/strings.xml
@@ -16,5 +16,5 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="cmas_presidential_level_alert" msgid="6968323372438785870">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‎‎‎‏‎‏‏‎‏‎‎‎‏‏‏‎‏‎‏‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‏‎‎‎‏‎‎‎‎‎‎‏‏‏‏‎‏‎‎‏‏‏‎‎Critical emergency alert‎‏‎‎‏‎"</string>
+    <string name="cmas_presidential_level_alert" msgid="6968323372438785870">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‎‎‎‏‎‏‏‎‏‎‎‎‏‏‏‎‏‎‏‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‏‎‎‎‏‎‎‎‎‎‎‏‏‏‏‎‏‎‎‏‏‏‎‎Critical emergency alert‎‏‎‎‏‎"</string>
 </resources>
diff --git a/res/values-mcc450-gl/strings.xml b/res/values-mcc450-gl/strings.xml
index 94e21c09..dae1c915 100644
--- a/res/values-mcc450-gl/strings.xml
+++ b/res/values-mcc450-gl/strings.xml
@@ -16,5 +16,5 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="cmas_presidential_level_alert" msgid="6968323372438785870">"Alerta de urxencia crítica"</string>
+    <string name="cmas_presidential_level_alert" msgid="6968323372438785870">"Alerta de emerxencia crítica"</string>
 </resources>
diff --git a/res/values-mcc466-en-rXC/strings.xml b/res/values-mcc466-en-rXC/strings.xml
index e957a1ba..8858bcad 100644
--- a/res/values-mcc466-en-rXC/strings.xml
+++ b/res/values-mcc466-en-rXC/strings.xml
@@ -16,5 +16,5 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="public_safety_message" msgid="4052407641316309322">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‎‎‎‎‏‎‏‏‎‏‎‎‎‏‏‏‎‏‏‏‏‎‎‏‏‎‏‏‏‏‎‎‏‎‏‏‏‎‎‏‎‏‎‎‏‎‏‎‎Alert message‎‏‎‎‏‎"</string>
+    <string name="public_safety_message" msgid="4052407641316309322">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‎‎‎‎‏‎‏‏‎‏‎‎‎‏‏‏‎‏‏‏‏‎‎‏‏‎‏‏‏‏‎‎‏‎‏‏‏‎‎‏‎‏‎‎‏‎‏‎‎Alert message‎‏‎‎‏‎"</string>
 </resources>
diff --git a/res/values-mcc716-en-rXC/strings.xml b/res/values-mcc716-en-rXC/strings.xml
index a41c3643..7a2bd721 100644
--- a/res/values-mcc716-en-rXC/strings.xml
+++ b/res/values-mcc716-en-rXC/strings.xml
@@ -16,6 +16,6 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="public_safety_message" msgid="7237115648380675159">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‎‎‎‏‏‎‏‏‏‏‎‏‏‎‎‏‏‎‎‏‎‏‎‏‏‎‏‎‎‏‏‏‎‎‎‎‏‎‎‏‏‎‏‎‏‎‎‏‎‎‎‏‎‏‎‏‏‏‎Informative message‎‏‎‎‏‎"</string>
-    <string name="enable_public_safety_messages_title" msgid="4037404034274507696">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‎‎‎‎‎‎‎‏‏‏‏‎‏‏‏‏‎‏‏‎‎‏‎‎‏‏‏‏‏‏‏‏‎‏‏‎‏‎‏‎‎‏‎‎‏‏‏‎‏‏‏‎‏‏‎‎‎‎‎Informative message‎‏‎‎‏‎"</string>
+    <string name="public_safety_message" msgid="7237115648380675159">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‏‏‏‎‎‏‎‎‎‏‏‎‏‏‏‏‎‏‏‎‎‏‏‎‎‏‎‏‎‏‏‎‏‎‎‏‏‏‎‎‎‎‏‎‎‏‏‎‏‎‏‎‎‏‎‎‎‏‎‏‎‏‏‏‎Informative message‎‏‎‎‏‎"</string>
+    <string name="enable_public_safety_messages_title" msgid="4037404034274507696">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‏‎‏‏‏‎‎‎‎‎‎‎‎‏‏‏‏‎‏‏‏‏‎‏‏‎‎‏‎‎‏‏‏‏‏‏‏‏‎‏‏‎‏‎‏‎‎‏‎‎‏‏‏‎‏‏‏‎‏‏‎‎‎‎‎Informative message‎‏‎‎‏‎"</string>
 </resources>
diff --git a/res/values-nl/strings.xml b/res/values-nl/strings.xml
index 2b501ccc..85e10c83 100644
--- a/res/values-nl/strings.xml
+++ b/res/values-nl/strings.xml
@@ -129,5 +129,5 @@
     <item msgid="3863339891188103437">"Elke 15 minuten"</item>
     <item msgid="6868848414437854609">"Uit"</item>
   </string-array>
-    <string name="emergency_alert_settings_title_watches" msgid="4477073412799894883">"Noodmeldingen voor draadloos"</string>
+    <string name="emergency_alert_settings_title_watches" msgid="4477073412799894883">"Noodwaarschuwingen"</string>
 </resources>
diff --git a/src/com/android/cellbroadcastreceiver/CellBroadcastAlertAudio.java b/src/com/android/cellbroadcastreceiver/CellBroadcastAlertAudio.java
index abb3235c..e75d09b1 100644
--- a/src/com/android/cellbroadcastreceiver/CellBroadcastAlertAudio.java
+++ b/src/com/android/cellbroadcastreceiver/CellBroadcastAlertAudio.java
@@ -349,6 +349,8 @@ public class CellBroadcastAlertAudio extends Service implements TextToSpeech.OnI
         // Vibration duration in milliseconds
         long vibrateDuration = 0;
 
+        // Get the alert tone duration. Negative tone duration value means we only play the tone
+        // once, not repeat it.
         int customAlertDuration = getResources().getInteger(R.integer.alert_duration);
 
         // Start the vibration first.
@@ -365,8 +367,12 @@ public class CellBroadcastAlertAudio extends Service implements TextToSpeech.OnI
             AudioAttributes.Builder attrBuilder = new AudioAttributes.Builder();
             attrBuilder.setUsage(AudioAttributes.USAGE_ALARM);
             AudioAttributes attr = attrBuilder.build();
-            VibrationEffect effect = VibrationEffect.createWaveform(vibrationPattern, 0);
-            log("vibrate: effect=" + effect + ", attr=" + attr);
+            // If we only play the tone once, then we also play the vibration pattern once.
+            int repeatIndex = (customAlertDuration < 0)
+                    ? -1 /* not repeat */ : 0 /* index to repeat */;
+            VibrationEffect effect = VibrationEffect.createWaveform(vibrationPattern, repeatIndex);
+            log("vibrate: effect=" + effect + ", attr=" + attr + ", duration="
+                    + customAlertDuration);
             mVibrator.vibrate(effect, attr);
         }
 
diff --git a/src/com/android/cellbroadcastreceiver/CellBroadcastAlertService.java b/src/com/android/cellbroadcastreceiver/CellBroadcastAlertService.java
index c779043a..7186ab22 100644
--- a/src/com/android/cellbroadcastreceiver/CellBroadcastAlertService.java
+++ b/src/com/android/cellbroadcastreceiver/CellBroadcastAlertService.java
@@ -33,6 +33,7 @@ import android.os.Binder;
 import android.os.Bundle;
 import android.os.IBinder;
 import android.os.PersistableBundle;
+import android.os.PowerManager;
 import android.os.RemoteException;
 import android.os.SystemClock;
 import android.os.UserHandle;
@@ -606,6 +607,11 @@ public class CellBroadcastAlertService extends Service {
             Intent alertDialogIntent = createDisplayMessageIntent(this,
                     CellBroadcastAlertDialog.class, messageList);
             alertDialogIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+            PowerManager pm = (PowerManager) getSystemService(Context.POWER_SERVICE);
+            // Wake up the device up regardless the scenario. (The device might be
+            // in screen saver mode that needs to be waken up otherwise the alert
+            // window can not be displayed.)
+            pm.wakeUp(SystemClock.uptimeMillis());
             startActivity(alertDialogIntent);
         }
 
-- 
2.17.1

